[2020-08-22 21:38:29,477][base.py:440][INFO][12248] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 21:38:29,480][base.py:881][INFO][12248] - Added job "time_task" to job store "default"
[2020-08-22 21:38:29,480][base.py:166][INFO][12248] - Scheduler started
[2020-08-22 21:38:29,480][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:38:29,480][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:39:29.477327+08:00 (in 59.997001 seconds)
[2020-08-22 21:38:29,531][_internal.py:113][INFO][12248] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 21:39:29,480][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:39:29,486][base.py:123][INFO][13712] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:39:29 CST)" (scheduled at 2020-08-22 21:39:29.477327+08:00)
[2020-08-22 21:39:29,487][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:40:29.477327+08:00 (in 59.992091 seconds)
[2020-08-22 21:39:29,488][base.py:144][INFO][13712] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:40:29 CST)" executed successfully
[2020-08-22 21:40:29,482][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:40:29,483][base.py:123][INFO][13712] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:40:29 CST)" (scheduled at 2020-08-22 21:40:29.477327+08:00)
[2020-08-22 21:40:29,484][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:41:29.477327+08:00 (in 59.994386 seconds)
[2020-08-22 21:40:29,484][base.py:144][INFO][13712] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:41:29 CST)" executed successfully
[2020-08-22 21:41:29,481][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:41:29,482][base.py:123][INFO][1052] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:41:29 CST)" (scheduled at 2020-08-22 21:41:29.477327+08:00)
[2020-08-22 21:41:29,483][base.py:144][INFO][1052] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:41:29 CST)" executed successfully
[2020-08-22 21:41:29,484][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:42:29.477327+08:00 (in 59.995801 seconds)
[2020-08-22 21:42:29,480][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:42:29,481][base.py:123][INFO][13712] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:42:29 CST)" (scheduled at 2020-08-22 21:42:29.477327+08:00)
[2020-08-22 21:42:29,482][base.py:144][INFO][13712] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:42:29 CST)" executed successfully
[2020-08-22 21:42:29,483][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:43:29.477327+08:00 (in 59.996537 seconds)
[2020-08-22 21:43:29,481][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:43:29,482][base.py:123][INFO][1052] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:43:29 CST)" (scheduled at 2020-08-22 21:43:29.477327+08:00)
[2020-08-22 21:43:29,483][base.py:144][INFO][1052] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:43:29 CST)" executed successfully
[2020-08-22 21:43:29,484][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:44:29.477327+08:00 (in 59.995734 seconds)
[2020-08-22 21:44:29,482][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:44:29,483][base.py:123][INFO][13836] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:44:29 CST)" (scheduled at 2020-08-22 21:44:29.477327+08:00)
[2020-08-22 21:44:29,484][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:45:29.477327+08:00 (in 59.995087 seconds)
[2020-08-22 21:44:29,485][base.py:144][INFO][13836] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:45:29 CST)" executed successfully
[2020-08-22 21:45:29,480][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:45:29,481][base.py:123][INFO][13712] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:45:29 CST)" (scheduled at 2020-08-22 21:45:29.477327+08:00)
[2020-08-22 21:45:29,482][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:46:29.477327+08:00 (in 59.996587 seconds)
[2020-08-22 21:45:29,483][base.py:144][INFO][13712] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:46:29 CST)" executed successfully
[2020-08-22 21:46:29,481][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:46:29,482][base.py:123][INFO][1052] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:46:29 CST)" (scheduled at 2020-08-22 21:46:29.477327+08:00)
[2020-08-22 21:46:29,483][base.py:144][INFO][1052] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:46:29 CST)" executed successfully
[2020-08-22 21:46:29,484][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:47:29.477327+08:00 (in 59.995954 seconds)
[2020-08-22 21:47:29,480][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:47:29,481][base.py:123][INFO][4800] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:47:29 CST)" (scheduled at 2020-08-22 21:47:29.477327+08:00)
[2020-08-22 21:47:29,481][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:48:29.477327+08:00 (in 59.996621 seconds)
[2020-08-22 21:47:29,482][base.py:144][INFO][4800] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:48:29 CST)" executed successfully
[2020-08-22 21:48:29,481][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:48:29,481][base.py:123][INFO][6624] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:48:29 CST)" (scheduled at 2020-08-22 21:48:29.477327+08:00)
[2020-08-22 21:48:29,481][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:49:29.477327+08:00 (in 59.996002 seconds)
[2020-08-22 21:48:29,481][base.py:144][INFO][6624] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:49:29 CST)" executed successfully
[2020-08-22 21:49:29,478][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:49:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:50:29.477327+08:00 (in 59.998415 seconds)
[2020-08-22 21:49:29,479][base.py:123][INFO][13712] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:50:29 CST)" (scheduled at 2020-08-22 21:49:29.477327+08:00)
[2020-08-22 21:49:29,480][base.py:144][INFO][13712] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:50:29 CST)" executed successfully
[2020-08-22 21:50:29,479][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:50:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:51:29.477327+08:00 (in 59.997462 seconds)
[2020-08-22 21:50:29,480][base.py:123][INFO][4572] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:51:29 CST)" (scheduled at 2020-08-22 21:50:29.477327+08:00)
[2020-08-22 21:50:29,480][base.py:144][INFO][4572] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:51:29 CST)" executed successfully
[2020-08-22 21:51:29,478][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:51:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:52:29.477327+08:00 (in 59.999192 seconds)
[2020-08-22 21:51:29,479][base.py:123][INFO][13836] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:52:29 CST)" (scheduled at 2020-08-22 21:51:29.477327+08:00)
[2020-08-22 21:51:29,480][base.py:144][INFO][13836] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:52:29 CST)" executed successfully
[2020-08-22 21:52:29,479][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:52:29,480][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:53:29.477327+08:00 (in 59.997970 seconds)
[2020-08-22 21:52:29,480][base.py:123][INFO][4800] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:53:29 CST)" (scheduled at 2020-08-22 21:52:29.477327+08:00)
[2020-08-22 21:52:29,481][base.py:144][INFO][4800] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:53:29 CST)" executed successfully
[2020-08-22 21:53:29,478][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:53:29,478][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:54:29.477327+08:00 (in 59.998364 seconds)
[2020-08-22 21:53:29,479][base.py:123][INFO][10644] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:54:29 CST)" (scheduled at 2020-08-22 21:53:29.477327+08:00)
[2020-08-22 21:53:29,479][base.py:144][INFO][10644] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:54:29 CST)" executed successfully
[2020-08-22 21:54:29,478][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:54:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:55:29.477327+08:00 (in 59.998851 seconds)
[2020-08-22 21:54:29,479][base.py:123][INFO][1052] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:55:29 CST)" (scheduled at 2020-08-22 21:54:29.477327+08:00)
[2020-08-22 21:54:29,480][base.py:144][INFO][1052] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:55:29 CST)" executed successfully
[2020-08-22 21:55:29,479][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:55:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:56:29.477327+08:00 (in 59.998318 seconds)
[2020-08-22 21:55:29,479][base.py:123][INFO][7276] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:56:29 CST)" (scheduled at 2020-08-22 21:55:29.477327+08:00)
[2020-08-22 21:55:29,481][base.py:144][INFO][7276] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:56:29 CST)" executed successfully
[2020-08-22 21:56:29,479][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:56:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:57:29.477327+08:00 (in 59.997440 seconds)
[2020-08-22 21:56:29,480][base.py:123][INFO][8672] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:57:29 CST)" (scheduled at 2020-08-22 21:56:29.477327+08:00)
[2020-08-22 21:56:29,480][base.py:144][INFO][8672] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:57:29 CST)" executed successfully
[2020-08-22 21:57:29,479][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:57:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:58:29.477327+08:00 (in 59.997471 seconds)
[2020-08-22 21:57:29,479][base.py:123][INFO][8456] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:58:29 CST)" (scheduled at 2020-08-22 21:57:29.477327+08:00)
[2020-08-22 21:57:29,480][base.py:144][INFO][8456] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:58:29 CST)" executed successfully
[2020-08-22 21:57:58,412][_internal.py:113][INFO][11316] - 127.0.0.1 - - [22/Aug/2020 21:57:58] "[31m[1mGET /v1/user/login HTTP/1.1[0m" 401 -
[2020-08-22 21:58:19,006][_internal.py:113][INFO][12868] - 127.0.0.1 - - [22/Aug/2020 21:58:19] "[37mPOST /v1/user/getToken HTTP/1.1[0m" 200 -
[2020-08-22 21:58:29,478][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:58:29,479][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 21:59:29.477327+08:00 (in 59.998841 seconds)
[2020-08-22 21:58:29,479][base.py:123][INFO][10644] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:59:29 CST)" (scheduled at 2020-08-22 21:58:29.477327+08:00)
[2020-08-22 21:58:29,480][base.py:144][INFO][10644] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 21:59:29 CST)" executed successfully
[2020-08-22 21:58:37,388][_internal.py:113][INFO][10688] - 127.0.0.1 - - [22/Aug/2020 21:58:37] "[31m[1mGET /v1/user/login HTTP/1.1[0m" 401 -
[2020-08-22 21:58:48,312][_internal.py:113][INFO][12480] - 127.0.0.1 - - [22/Aug/2020 21:58:48] "[31m[1mGET /v1/user/login HTTP/1.1[0m" 401 -
[2020-08-22 21:58:53,011][app.py:1892][ERROR][5012] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    user=User.objects(account=UserTokenInfo.account).first()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\manager.py", line 37, in __get__
    queryset = queryset_class(owner, owner._get_collection())
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\document.py", line 207, in _get_collection
    db = cls._get_db()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\document.py", line 185, in _get_db
    return get_db(cls._meta.get("db_alias", DEFAULT_CONNECTION_NAME))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\connection.py", line 356, in get_db
    conn = get_connection(alias)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\connection.py", line 268, in get_connection
    raise ConnectionFailure(msg)
mongoengine.connection.ConnectionFailure: Connection with alias "userInfo" has not been defined
[2020-08-22 21:58:53,023][_internal.py:113][INFO][5012] - 127.0.0.1 - - [22/Aug/2020 21:58:53] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 21:59:29,478][base.py:940][DEBUG][1692] - Looking for jobs to run
[2020-08-22 21:59:29,478][base.py:1020][DEBUG][1692] - Next wakeup is due at 2020-08-22 22:00:29.477327+08:00 (in 59.998422 seconds)
[2020-08-22 21:59:29,478][base.py:123][INFO][13712] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:00:29 CST)" (scheduled at 2020-08-22 21:59:29.477327+08:00)
[2020-08-22 21:59:29,479][base.py:144][INFO][13712] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:00:29 CST)" executed successfully
[2020-08-22 21:59:49,864][base.py:440][INFO][2092] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 21:59:49,865][base.py:881][INFO][2092] - Added job "time_task" to job store "default"
[2020-08-22 21:59:49,865][base.py:166][INFO][2092] - Scheduler started
[2020-08-22 21:59:49,866][base.py:940][DEBUG][3504] - Looking for jobs to run
[2020-08-22 21:59:49,866][base.py:1020][DEBUG][3504] - Next wakeup is due at 2020-08-22 22:00:49.864697+08:00 (in 59.998000 seconds)
[2020-08-22 21:59:49,895][_internal.py:113][INFO][2092] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:00:17,952][base.py:440][INFO][12608] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:00:17,955][base.py:881][INFO][12608] - Added job "time_task" to job store "default"
[2020-08-22 22:00:17,955][base.py:166][INFO][12608] - Scheduler started
[2020-08-22 22:00:17,963][base.py:940][DEBUG][4496] - Looking for jobs to run
[2020-08-22 22:00:17,964][base.py:1020][DEBUG][4496] - Next wakeup is due at 2020-08-22 22:01:17.952161+08:00 (in 59.988020 seconds)
[2020-08-22 22:00:18,019][_internal.py:113][INFO][12608] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:01:06,232][app.py:1892][ERROR][7264] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    user=User.objects(account=UserTokenInfo.account).first()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\manager.py", line 37, in __get__
    queryset = queryset_class(owner, owner._get_collection())
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\document.py", line 207, in _get_collection
    db = cls._get_db()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\document.py", line 185, in _get_db
    return get_db(cls._meta.get("db_alias", DEFAULT_CONNECTION_NAME))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\connection.py", line 356, in get_db
    conn = get_connection(alias)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\connection.py", line 268, in get_connection
    raise ConnectionFailure(msg)
mongoengine.connection.ConnectionFailure: Connection with alias "userInfo" has not been defined
[2020-08-22 22:01:07,873][_internal.py:113][INFO][7264] - 127.0.0.1 - - [22/Aug/2020 22:01:07] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:01:17,964][base.py:940][DEBUG][4496] - Looking for jobs to run
[2020-08-22 22:01:17,968][base.py:123][INFO][6692] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:01:17 CST)" (scheduled at 2020-08-22 22:01:17.952161+08:00)
[2020-08-22 22:01:17,971][base.py:1020][DEBUG][4496] - Next wakeup is due at 2020-08-22 22:02:17.952161+08:00 (in 59.987743 seconds)
[2020-08-22 22:01:17,972][base.py:144][INFO][6692] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:02:17 CST)" executed successfully
[2020-08-22 22:01:31,983][base.py:440][INFO][10740] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:01:31,985][base.py:881][INFO][10740] - Added job "time_task" to job store "default"
[2020-08-22 22:01:31,985][base.py:166][INFO][10740] - Scheduler started
[2020-08-22 22:01:31,987][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:01:31,988][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:02:31.983810+08:00 (in 59.994999 seconds)
[2020-08-22 22:01:32,070][_internal.py:113][INFO][10740] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:01:36,809][app.py:1892][ERROR][12208] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    user=User.objects(account=UserTokenInfo.account).first()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\manager.py", line 37, in __get__
    queryset = queryset_class(owner, owner._get_collection())
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\document.py", line 207, in _get_collection
    db = cls._get_db()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\document.py", line 185, in _get_db
    return get_db(cls._meta.get("db_alias", DEFAULT_CONNECTION_NAME))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\connection.py", line 356, in get_db
    conn = get_connection(alias)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\connection.py", line 268, in get_connection
    raise ConnectionFailure(msg)
mongoengine.connection.ConnectionFailure: Connection with alias "userInfo" has not been defined
[2020-08-22 22:01:36,815][_internal.py:113][INFO][12208] - 127.0.0.1 - - [22/Aug/2020 22:01:36] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:02:31,985][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:02:31,990][base.py:123][INFO][14176] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:02:31 CST)" (scheduled at 2020-08-22 22:02:31.983810+08:00)
[2020-08-22 22:02:31,994][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:03:31.983810+08:00 (in 59.997338 seconds)
[2020-08-22 22:02:31,995][base.py:144][INFO][14176] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:03:31 CST)" executed successfully
[2020-08-22 22:03:31,997][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:03:32,000][base.py:123][INFO][14176] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:03:31 CST)" (scheduled at 2020-08-22 22:03:31.983810+08:00)
[2020-08-22 22:03:32,004][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:04:31.983810+08:00 (in 59.985721 seconds)
[2020-08-22 22:03:32,005][base.py:144][INFO][14176] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:04:31 CST)" executed successfully
[2020-08-22 22:04:31,992][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:04:31,996][base.py:123][INFO][7424] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:04:31 CST)" (scheduled at 2020-08-22 22:04:31.983810+08:00)
[2020-08-22 22:04:32,002][base.py:144][INFO][7424] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:04:31 CST)" executed successfully
[2020-08-22 22:04:32,005][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:05:31.983810+08:00 (in 59.990281 seconds)
[2020-08-22 22:05:32,015][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:05:32,019][base.py:123][INFO][14176] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:05:31 CST)" (scheduled at 2020-08-22 22:05:31.983810+08:00)
[2020-08-22 22:05:32,021][base.py:144][INFO][14176] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:05:31 CST)" executed successfully
[2020-08-22 22:05:32,024][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:06:31.983810+08:00 (in 59.966807 seconds)
[2020-08-22 22:06:31,993][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:06:31,994][base.py:123][INFO][7424] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:06:31 CST)" (scheduled at 2020-08-22 22:06:31.983810+08:00)
[2020-08-22 22:06:31,995][base.py:144][INFO][7424] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:06:31 CST)" executed successfully
[2020-08-22 22:06:31,996][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:07:31.983810+08:00 (in 59.990362 seconds)
[2020-08-22 22:07:31,993][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:07:31,994][base.py:123][INFO][14176] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:07:31 CST)" (scheduled at 2020-08-22 22:07:31.983810+08:00)
[2020-08-22 22:07:31,996][base.py:144][INFO][14176] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:07:31 CST)" executed successfully
[2020-08-22 22:07:31,998][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:08:31.983810+08:00 (in 59.990004 seconds)
[2020-08-22 22:08:31,997][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:08:32,000][base.py:123][INFO][7424] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:08:31 CST)" (scheduled at 2020-08-22 22:08:31.983810+08:00)
[2020-08-22 22:08:32,003][base.py:144][INFO][7424] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:08:31 CST)" executed successfully
[2020-08-22 22:08:32,007][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:09:31.983810+08:00 (in 59.984883 seconds)
[2020-08-22 22:09:32,008][base.py:940][DEBUG][360] - Looking for jobs to run
[2020-08-22 22:09:32,011][base.py:123][INFO][8676] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:09:31 CST)" (scheduled at 2020-08-22 22:09:31.983810+08:00)
[2020-08-22 22:09:32,014][base.py:144][INFO][8676] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:09:31 CST)" executed successfully
[2020-08-22 22:09:32,015][base.py:1020][DEBUG][360] - Next wakeup is due at 2020-08-22 22:10:31.983810+08:00 (in 59.974375 seconds)
[2020-08-22 22:10:26,367][base.py:440][INFO][5612] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:10:26,368][base.py:881][INFO][5612] - Added job "time_task" to job store "default"
[2020-08-22 22:10:26,368][base.py:166][INFO][5612] - Scheduler started
[2020-08-22 22:10:26,369][base.py:940][DEBUG][8252] - Looking for jobs to run
[2020-08-22 22:10:26,369][base.py:1020][DEBUG][8252] - Next wakeup is due at 2020-08-22 22:11:26.367203+08:00 (in 59.998001 seconds)
[2020-08-22 22:10:26,400][_internal.py:113][INFO][5612] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:10:34,856][app.py:1892][ERROR][11712] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    user=User.objects(account=UserTokenInfo.account).first()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 287, in first
    result = queryset[0]
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 191, in __getitem__
    queryset._cursor[key],
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 643, in __getitem__
    clone = self.clone()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 274, in clone
    return self._clone(True)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 294, in _clone
    data = self._deepcopy(data)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1259, in _deepcopy
    value = self._deepcopy(value, memo)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1261, in _deepcopy
    value = copy.deepcopy(value, memo)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\copy.py", line 169, in deepcopy
    rv = reductor(4)
TypeError: can't pickle property objects
[2020-08-22 22:10:34,873][_internal.py:113][INFO][11712] - 127.0.0.1 - - [22/Aug/2020 22:10:34] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:11:18,006][base.py:440][INFO][6288] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:11:18,007][base.py:881][INFO][6288] - Added job "time_task" to job store "default"
[2020-08-22 22:11:18,007][base.py:166][INFO][6288] - Scheduler started
[2020-08-22 22:11:18,008][base.py:940][DEBUG][13192] - Looking for jobs to run
[2020-08-22 22:11:18,008][base.py:1020][DEBUG][13192] - Next wakeup is due at 2020-08-22 22:12:18.006888+08:00 (in 59.997987 seconds)
[2020-08-22 22:11:18,044][_internal.py:113][INFO][6288] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:11:20,277][app.py:1892][ERROR][8496] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    user=User.objects(account=UserTokenInfo.account).first()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 287, in first
    result = queryset[0]
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 191, in __getitem__
    queryset._cursor[key],
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 643, in __getitem__
    clone = self.clone()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 274, in clone
    return self._clone(True)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 294, in _clone
    data = self._deepcopy(data)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1259, in _deepcopy
    value = self._deepcopy(value, memo)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1261, in _deepcopy
    value = copy.deepcopy(value, memo)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\copy.py", line 169, in deepcopy
    rv = reductor(4)
TypeError: can't pickle property objects
[2020-08-22 22:11:20,284][_internal.py:113][INFO][8496] - 127.0.0.1 - - [22/Aug/2020 22:11:20] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:11:33,045][base.py:440][INFO][5388] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:11:33,048][base.py:881][INFO][5388] - Added job "time_task" to job store "default"
[2020-08-22 22:11:33,048][base.py:166][INFO][5388] - Scheduler started
[2020-08-22 22:11:33,052][base.py:940][DEBUG][2636] - Looking for jobs to run
[2020-08-22 22:11:33,053][base.py:1020][DEBUG][2636] - Next wakeup is due at 2020-08-22 22:12:33.044032+08:00 (in 59.991006 seconds)
[2020-08-22 22:11:33,112][_internal.py:113][INFO][5388] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:11:48,931][app.py:1892][ERROR][11788] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    user=User.objects(account=UserTokenInfo.account).first()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 287, in first
    result = queryset[0]
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 191, in __getitem__
    queryset._cursor[key],
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 643, in __getitem__
    clone = self.clone()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 274, in clone
    return self._clone(True)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 294, in _clone
    data = self._deepcopy(data)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1259, in _deepcopy
    value = self._deepcopy(value, memo)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1261, in _deepcopy
    value = copy.deepcopy(value, memo)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\copy.py", line 169, in deepcopy
    rv = reductor(4)
TypeError: can't pickle property objects
[2020-08-22 22:11:48,937][_internal.py:113][INFO][11788] - 127.0.0.1 - - [22/Aug/2020 22:11:48] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:11:58,757][base.py:440][INFO][9252] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:11:58,758][base.py:881][INFO][9252] - Added job "time_task" to job store "default"
[2020-08-22 22:11:58,758][base.py:166][INFO][9252] - Scheduler started
[2020-08-22 22:11:58,758][base.py:940][DEBUG][14004] - Looking for jobs to run
[2020-08-22 22:11:58,758][base.py:1020][DEBUG][14004] - Next wakeup is due at 2020-08-22 22:12:58.757976+08:00 (in 59.999000 seconds)
[2020-08-22 22:11:58,789][_internal.py:113][INFO][9252] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:12:04,131][app.py:1892][ERROR][12352] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 28, in verify_token
    if user:
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 208, in __bool__
    return self._has_data()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 204, in _has_data
    return False if queryset.first() is None else True
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 287, in first
    result = queryset[0]
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 193, in __getitem__
    only_fields=self.only_fields,
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\base\document.py", line 784, in _from_son
    __auto_convert=False, _created=created, __only_fields=only_fields, **data
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\base\document.py", line 99, in __init__
    raise FieldDoesNotExist(msg)
mongoengine.errors.FieldDoesNotExist: The fields "{'password'}" do not exist on the document "User"
[2020-08-22 22:12:04,142][_internal.py:113][INFO][12352] - 127.0.0.1 - - [22/Aug/2020 22:12:04] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:12:32,826][base.py:440][INFO][1628] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:12:32,828][base.py:881][INFO][1628] - Added job "time_task" to job store "default"
[2020-08-22 22:12:32,828][base.py:166][INFO][1628] - Scheduler started
[2020-08-22 22:12:32,828][base.py:940][DEBUG][8988] - Looking for jobs to run
[2020-08-22 22:12:32,828][base.py:1020][DEBUG][8988] - Next wakeup is due at 2020-08-22 22:13:32.826075+08:00 (in 59.998005 seconds)
[2020-08-22 22:12:32,858][_internal.py:113][INFO][1628] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:12:35,659][app.py:1892][ERROR][13988] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    user=User.objects(account=UserTokenInfo.account).first()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 287, in first
    result = queryset[0]
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\mongoengine\queryset\base.py", line 191, in __getitem__
    queryset._cursor[key],
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 643, in __getitem__
    clone = self.clone()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 274, in clone
    return self._clone(True)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 294, in _clone
    data = self._deepcopy(data)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1259, in _deepcopy
    value = self._deepcopy(value, memo)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\pymongo\cursor.py", line 1261, in _deepcopy
    value = copy.deepcopy(value, memo)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\copy.py", line 169, in deepcopy
    rv = reductor(4)
TypeError: can't pickle property objects
[2020-08-22 22:12:35,665][_internal.py:113][INFO][13988] - 127.0.0.1 - - [22/Aug/2020 22:12:35] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:12:50,430][base.py:440][INFO][2764] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:12:50,431][base.py:881][INFO][2764] - Added job "time_task" to job store "default"
[2020-08-22 22:12:50,431][base.py:166][INFO][2764] - Scheduler started
[2020-08-22 22:12:50,432][base.py:940][DEBUG][8156] - Looking for jobs to run
[2020-08-22 22:12:50,432][base.py:1020][DEBUG][8156] - Next wakeup is due at 2020-08-22 22:13:50.430988+08:00 (in 59.998001 seconds)
[2020-08-22 22:12:50,464][_internal.py:113][INFO][2764] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:12:56,514][_internal.py:113][INFO][3036] - 127.0.0.1 - - [22/Aug/2020 22:12:56] "[37mGET /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 22:13:30,831][base.py:440][INFO][7032] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:13:30,833][base.py:881][INFO][7032] - Added job "time_task" to job store "default"
[2020-08-22 22:13:30,834][base.py:166][INFO][7032] - Scheduler started
[2020-08-22 22:13:30,835][base.py:940][DEBUG][9268] - Looking for jobs to run
[2020-08-22 22:13:30,836][base.py:1020][DEBUG][9268] - Next wakeup is due at 2020-08-22 22:14:30.831204+08:00 (in 59.995004 seconds)
[2020-08-22 22:13:30,901][_internal.py:113][INFO][7032] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:13:57,039][base.py:440][INFO][2620] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:13:57,041][base.py:881][INFO][2620] - Added job "time_task" to job store "default"
[2020-08-22 22:13:57,041][base.py:166][INFO][2620] - Scheduler started
[2020-08-22 22:13:57,043][base.py:940][DEBUG][5872] - Looking for jobs to run
[2020-08-22 22:13:57,044][base.py:1020][DEBUG][5872] - Next wakeup is due at 2020-08-22 22:14:57.038902+08:00 (in 59.995002 seconds)
[2020-08-22 22:13:57,097][_internal.py:113][INFO][2620] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:14:03,136][base.py:440][INFO][3524] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:14:03,137][base.py:881][INFO][3524] - Added job "time_task" to job store "default"
[2020-08-22 22:14:03,137][base.py:166][INFO][3524] - Scheduler started
[2020-08-22 22:14:03,138][base.py:940][DEBUG][2944] - Looking for jobs to run
[2020-08-22 22:14:03,138][base.py:1020][DEBUG][2944] - Next wakeup is due at 2020-08-22 22:15:03.136515+08:00 (in 59.998001 seconds)
[2020-08-22 22:14:03,169][_internal.py:113][INFO][3524] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:14:07,383][base.py:440][INFO][2040] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:14:07,387][base.py:881][INFO][2040] - Added job "time_task" to job store "default"
[2020-08-22 22:14:07,387][base.py:166][INFO][2040] - Scheduler started
[2020-08-22 22:14:07,390][base.py:940][DEBUG][8920] - Looking for jobs to run
[2020-08-22 22:14:07,392][base.py:1020][DEBUG][8920] - Next wakeup is due at 2020-08-22 22:15:07.382260+08:00 (in 59.991005 seconds)
[2020-08-22 22:14:07,458][_internal.py:113][INFO][2040] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:15:07,385][base.py:940][DEBUG][8920] - Looking for jobs to run
[2020-08-22 22:15:07,388][base.py:123][INFO][3228] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:15:07 CST)" (scheduled at 2020-08-22 22:15:07.382260+08:00)
[2020-08-22 22:15:07,389][base.py:1020][DEBUG][8920] - Next wakeup is due at 2020-08-22 22:16:07.382260+08:00 (in 59.995699 seconds)
[2020-08-22 22:15:07,391][base.py:144][INFO][3228] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:16:07 CST)" executed successfully
[2020-08-22 22:15:22,060][base.py:440][INFO][4508] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:15:22,061][base.py:881][INFO][4508] - Added job "time_task" to job store "default"
[2020-08-22 22:15:22,062][base.py:166][INFO][4508] - Scheduler started
[2020-08-22 22:15:22,062][base.py:940][DEBUG][9868] - Looking for jobs to run
[2020-08-22 22:15:22,062][base.py:1020][DEBUG][9868] - Next wakeup is due at 2020-08-22 22:16:22.060019+08:00 (in 59.998001 seconds)
[2020-08-22 22:15:22,099][_internal.py:113][INFO][4508] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:15:28,390][_internal.py:113][INFO][6624] - 127.0.0.1 - - [22/Aug/2020 22:15:28] "[37mGET /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 22:15:48,121][base.py:440][INFO][2944] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:15:48,122][base.py:881][INFO][2944] - Added job "time_task" to job store "default"
[2020-08-22 22:15:48,122][base.py:166][INFO][2944] - Scheduler started
[2020-08-22 22:15:48,123][base.py:940][DEBUG][12436] - Looking for jobs to run
[2020-08-22 22:15:48,123][base.py:1020][DEBUG][12436] - Next wakeup is due at 2020-08-22 22:16:48.121124+08:00 (in 59.998014 seconds)
[2020-08-22 22:15:48,153][_internal.py:113][INFO][2944] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:15:51,076][app.py:1892][ERROR][14144] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 27, in verify_token
    print(UserTokenInfo['account'])
TypeError: 'type' object is not subscriptable
[2020-08-22 22:15:51,078][_internal.py:113][INFO][14144] - 127.0.0.1 - - [22/Aug/2020 22:15:51] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:16:25,692][base.py:440][INFO][5152] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:16:25,693][base.py:881][INFO][5152] - Added job "time_task" to job store "default"
[2020-08-22 22:16:25,693][base.py:166][INFO][5152] - Scheduler started
[2020-08-22 22:16:25,694][base.py:940][DEBUG][9744] - Looking for jobs to run
[2020-08-22 22:16:25,694][base.py:1020][DEBUG][9744] - Next wakeup is due at 2020-08-22 22:17:25.692719+08:00 (in 59.998001 seconds)
[2020-08-22 22:16:25,737][_internal.py:113][INFO][5152] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:16:30,875][_internal.py:113][INFO][6560] - 127.0.0.1 - - [22/Aug/2020 22:16:30] "[37mGET /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 22:17:25,693][base.py:940][DEBUG][9744] - Looking for jobs to run
[2020-08-22 22:17:25,694][base.py:123][INFO][8568] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:17:25 CST)" (scheduled at 2020-08-22 22:17:25.692719+08:00)
[2020-08-22 22:17:25,694][base.py:144][INFO][8568] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:17:25 CST)" executed successfully
[2020-08-22 22:17:25,695][base.py:1020][DEBUG][9744] - Next wakeup is due at 2020-08-22 22:18:25.692719+08:00 (in 59.998880 seconds)
[2020-08-22 22:18:25,695][base.py:940][DEBUG][9744] - Looking for jobs to run
[2020-08-22 22:18:25,696][base.py:123][INFO][8568] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:18:25 CST)" (scheduled at 2020-08-22 22:18:25.692719+08:00)
[2020-08-22 22:18:25,697][base.py:1020][DEBUG][9744] - Next wakeup is due at 2020-08-22 22:19:25.692719+08:00 (in 59.997092 seconds)
[2020-08-22 22:18:25,698][base.py:144][INFO][8568] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:19:25 CST)" executed successfully
[2020-08-22 22:18:55,783][base.py:440][INFO][11392] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:18:55,784][base.py:881][INFO][11392] - Added job "time_task" to job store "default"
[2020-08-22 22:18:55,784][base.py:166][INFO][11392] - Scheduler started
[2020-08-22 22:18:55,785][base.py:940][DEBUG][8324] - Looking for jobs to run
[2020-08-22 22:18:55,785][base.py:1020][DEBUG][8324] - Next wakeup is due at 2020-08-22 22:19:55.783974+08:00 (in 59.998001 seconds)
[2020-08-22 22:18:55,818][_internal.py:113][INFO][11392] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:18:58,933][_internal.py:113][INFO][14112] - 127.0.0.1 - - [22/Aug/2020 22:18:58] "[37mGET /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 22:19:55,785][base.py:940][DEBUG][8324] - Looking for jobs to run
[2020-08-22 22:19:55,785][base.py:123][INFO][11060] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:19:55 CST)" (scheduled at 2020-08-22 22:19:55.783974+08:00)
[2020-08-22 22:19:55,785][base.py:144][INFO][11060] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:19:55 CST)" executed successfully
[2020-08-22 22:19:55,786][base.py:1020][DEBUG][8324] - Next wakeup is due at 2020-08-22 22:20:55.783974+08:00 (in 59.998948 seconds)
[2020-08-22 22:20:03,996][base.py:440][INFO][7768] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:20:03,997][base.py:881][INFO][7768] - Added job "time_task" to job store "default"
[2020-08-22 22:20:03,997][base.py:166][INFO][7768] - Scheduler started
[2020-08-22 22:20:03,997][base.py:940][DEBUG][10620] - Looking for jobs to run
[2020-08-22 22:20:03,997][base.py:1020][DEBUG][10620] - Next wakeup is due at 2020-08-22 22:21:03.995677+08:00 (in 59.998001 seconds)
[2020-08-22 22:20:04,025][_internal.py:113][INFO][7768] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:20:07,456][_internal.py:113][INFO][4048] - 127.0.0.1 - - [22/Aug/2020 22:20:07] "[37mGET /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 22:21:03,997][base.py:940][DEBUG][10620] - Looking for jobs to run
[2020-08-22 22:21:03,998][base.py:123][INFO][11416] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:21:03 CST)" (scheduled at 2020-08-22 22:21:03.995677+08:00)
[2020-08-22 22:21:03,998][base.py:144][INFO][11416] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:21:03 CST)" executed successfully
[2020-08-22 22:21:03,999][base.py:1020][DEBUG][10620] - Next wakeup is due at 2020-08-22 22:22:03.995677+08:00 (in 59.998289 seconds)
[2020-08-22 22:21:42,173][base.py:440][INFO][1848] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:21:42,174][base.py:881][INFO][1848] - Added job "time_task" to job store "default"
[2020-08-22 22:21:42,174][base.py:166][INFO][1848] - Scheduler started
[2020-08-22 22:21:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:21:42,175][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:22:42.173285+08:00 (in 59.998001 seconds)
[2020-08-22 22:21:42,205][_internal.py:113][INFO][1848] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:22:42,174][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:22:42,175][base.py:123][INFO][13332] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:22:42 CST)" (scheduled at 2020-08-22 22:22:42.173285+08:00)
[2020-08-22 22:22:42,175][base.py:144][INFO][13332] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:22:42 CST)" executed successfully
[2020-08-22 22:22:42,176][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:23:42.173285+08:00 (in 59.998839 seconds)
[2020-08-22 22:23:42,176][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:23:42,176][base.py:123][INFO][13332] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:23:42 CST)" (scheduled at 2020-08-22 22:23:42.173285+08:00)
[2020-08-22 22:23:42,176][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:24:42.173285+08:00 (in 59.997062 seconds)
[2020-08-22 22:23:42,177][base.py:144][INFO][13332] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:24:42 CST)" executed successfully
[2020-08-22 22:24:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:24:42,175][base.py:123][INFO][13964] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:24:42 CST)" (scheduled at 2020-08-22 22:24:42.173285+08:00)
[2020-08-22 22:24:42,176][base.py:144][INFO][13964] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:24:42 CST)" executed successfully
[2020-08-22 22:24:42,176][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:25:42.173285+08:00 (in 59.997705 seconds)
[2020-08-22 22:25:42,176][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:25:42,176][base.py:123][INFO][13964] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:25:42 CST)" (scheduled at 2020-08-22 22:25:42.173285+08:00)
[2020-08-22 22:25:42,176][base.py:144][INFO][13964] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:25:42 CST)" executed successfully
[2020-08-22 22:25:42,177][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:26:42.173285+08:00 (in 59.997237 seconds)
[2020-08-22 22:26:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:26:42,176][base.py:123][INFO][10016] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:26:42 CST)" (scheduled at 2020-08-22 22:26:42.173285+08:00)
[2020-08-22 22:26:42,177][base.py:144][INFO][10016] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:26:42 CST)" executed successfully
[2020-08-22 22:26:42,177][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:27:42.173285+08:00 (in 59.997645 seconds)
[2020-08-22 22:27:42,177][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:27:42,177][base.py:123][INFO][13268] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:27:42 CST)" (scheduled at 2020-08-22 22:27:42.173285+08:00)
[2020-08-22 22:27:42,177][base.py:144][INFO][13268] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:27:42 CST)" executed successfully
[2020-08-22 22:27:42,178][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:28:42.173285+08:00 (in 59.996166 seconds)
[2020-08-22 22:28:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:28:42,175][base.py:123][INFO][10016] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:28:42 CST)" (scheduled at 2020-08-22 22:28:42.173285+08:00)
[2020-08-22 22:28:42,175][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:29:42.173285+08:00 (in 59.998020 seconds)
[2020-08-22 22:28:42,175][base.py:144][INFO][10016] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:29:42 CST)" executed successfully
[2020-08-22 22:29:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:29:42,175][base.py:123][INFO][13332] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:29:42 CST)" (scheduled at 2020-08-22 22:29:42.173285+08:00)
[2020-08-22 22:29:42,176][base.py:144][INFO][13332] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:29:42 CST)" executed successfully
[2020-08-22 22:29:42,177][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:30:42.173285+08:00 (in 59.998158 seconds)
[2020-08-22 22:30:42,176][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:30:42,176][base.py:123][INFO][13964] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:30:42 CST)" (scheduled at 2020-08-22 22:30:42.173285+08:00)
[2020-08-22 22:30:42,176][base.py:144][INFO][13964] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:30:42 CST)" executed successfully
[2020-08-22 22:30:42,177][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:31:42.173285+08:00 (in 59.997157 seconds)
[2020-08-22 22:31:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:31:42,176][base.py:123][INFO][13280] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:31:42 CST)" (scheduled at 2020-08-22 22:31:42.173285+08:00)
[2020-08-22 22:31:42,176][base.py:144][INFO][13280] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:31:42 CST)" executed successfully
[2020-08-22 22:31:42,177][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:32:42.173285+08:00 (in 59.997502 seconds)
[2020-08-22 22:32:42,176][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:32:42,177][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:33:42.173285+08:00 (in 59.996585 seconds)
[2020-08-22 22:32:42,177][base.py:123][INFO][10016] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:33:42 CST)" (scheduled at 2020-08-22 22:32:42.173285+08:00)
[2020-08-22 22:32:42,178][base.py:144][INFO][10016] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:33:42 CST)" executed successfully
[2020-08-22 22:33:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:33:42,175][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:34:42.173285+08:00 (in 59.998119 seconds)
[2020-08-22 22:33:42,175][base.py:123][INFO][13268] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:34:42 CST)" (scheduled at 2020-08-22 22:33:42.173285+08:00)
[2020-08-22 22:33:42,175][base.py:144][INFO][13268] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:34:42 CST)" executed successfully
[2020-08-22 22:34:42,174][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:34:42,175][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:35:42.173285+08:00 (in 59.998583 seconds)
[2020-08-22 22:34:42,175][base.py:123][INFO][8336] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:35:42 CST)" (scheduled at 2020-08-22 22:34:42.173285+08:00)
[2020-08-22 22:34:42,176][base.py:144][INFO][8336] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:35:42 CST)" executed successfully
[2020-08-22 22:35:42,174][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:35:42,174][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:36:42.173285+08:00 (in 59.998486 seconds)
[2020-08-22 22:35:42,175][base.py:123][INFO][12436] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:36:42 CST)" (scheduled at 2020-08-22 22:35:42.173285+08:00)
[2020-08-22 22:35:42,175][base.py:144][INFO][12436] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:36:42 CST)" executed successfully
[2020-08-22 22:36:42,175][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:36:42,175][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:37:42.173285+08:00 (in 59.997631 seconds)
[2020-08-22 22:36:42,176][base.py:123][INFO][8044] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:37:42 CST)" (scheduled at 2020-08-22 22:36:42.173285+08:00)
[2020-08-22 22:36:42,176][base.py:144][INFO][8044] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:37:42 CST)" executed successfully
[2020-08-22 22:37:42,174][base.py:940][DEBUG][6740] - Looking for jobs to run
[2020-08-22 22:37:42,174][base.py:1020][DEBUG][6740] - Next wakeup is due at 2020-08-22 22:38:42.173285+08:00 (in 59.998286 seconds)
[2020-08-22 22:37:42,175][base.py:123][INFO][13332] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:38:42 CST)" (scheduled at 2020-08-22 22:37:42.173285+08:00)
[2020-08-22 22:37:42,175][base.py:144][INFO][13332] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:38:42 CST)" executed successfully
[2020-08-22 22:38:20,129][base.py:440][INFO][8340] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:38:20,129][base.py:881][INFO][8340] - Added job "time_task" to job store "default"
[2020-08-22 22:38:20,129][base.py:166][INFO][8340] - Scheduler started
[2020-08-22 22:38:20,129][base.py:940][DEBUG][2636] - Looking for jobs to run
[2020-08-22 22:38:20,130][base.py:1020][DEBUG][2636] - Next wakeup is due at 2020-08-22 22:39:20.129000+08:00 (in 59.998001 seconds)
[2020-08-22 22:38:20,162][_internal.py:113][INFO][8340] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:38:27,187][_internal.py:113][INFO][7628] - 127.0.0.1 - - [22/Aug/2020 22:38:27] "[35m[1mPOST /v1/user/getToken HTTP/1.1[0m" 500 -
[2020-08-22 22:38:38,692][base.py:440][INFO][8676] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:38:38,694][base.py:881][INFO][8676] - Added job "time_task" to job store "default"
[2020-08-22 22:38:38,694][base.py:166][INFO][8676] - Scheduler started
[2020-08-22 22:38:38,696][base.py:940][DEBUG][9736] - Looking for jobs to run
[2020-08-22 22:38:38,697][base.py:1020][DEBUG][9736] - Next wakeup is due at 2020-08-22 22:39:38.692141+08:00 (in 59.995003 seconds)
[2020-08-22 22:38:38,756][_internal.py:113][INFO][8676] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:39:38,706][base.py:940][DEBUG][9736] - Looking for jobs to run
[2020-08-22 22:39:38,714][base.py:123][INFO][7188] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:39:38 CST)" (scheduled at 2020-08-22 22:39:38.692141+08:00)
[2020-08-22 22:39:38,719][base.py:1020][DEBUG][9736] - Next wakeup is due at 2020-08-22 22:40:38.692141+08:00 (in 59.984752 seconds)
[2020-08-22 22:39:38,725][base.py:144][INFO][7188] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:40:38 CST)" executed successfully
[2020-08-22 22:39:44,221][_internal.py:113][INFO][12676] - 127.0.0.1 - - [22/Aug/2020 22:39:44] "[35m[1mPOST /v1/user/getToken HTTP/1.1[0m" 500 -
[2020-08-22 22:40:05,377][base.py:440][INFO][8600] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:40:05,377][base.py:881][INFO][8600] - Added job "time_task" to job store "default"
[2020-08-22 22:40:05,378][base.py:166][INFO][8600] - Scheduler started
[2020-08-22 22:40:05,378][base.py:940][DEBUG][12232] - Looking for jobs to run
[2020-08-22 22:40:05,378][base.py:1020][DEBUG][12232] - Next wakeup is due at 2020-08-22 22:41:05.376505+08:00 (in 59.998001 seconds)
[2020-08-22 22:40:05,409][_internal.py:113][INFO][8600] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:40:12,177][_internal.py:113][INFO][11468] - 127.0.0.1 - - [22/Aug/2020 22:40:12] "[31m[1mPOST /v1/user/getToken HTTP/1.1[0m" 400 -
[2020-08-22 22:41:05,377][base.py:940][DEBUG][12232] - Looking for jobs to run
[2020-08-22 22:41:05,378][base.py:123][INFO][1556] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:41:05 CST)" (scheduled at 2020-08-22 22:41:05.376505+08:00)
[2020-08-22 22:41:05,378][base.py:1020][DEBUG][12232] - Next wakeup is due at 2020-08-22 22:42:05.376505+08:00 (in 59.998580 seconds)
[2020-08-22 22:41:05,379][base.py:144][INFO][1556] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:42:05 CST)" executed successfully
[2020-08-22 22:42:05,379][base.py:940][DEBUG][12232] - Looking for jobs to run
[2020-08-22 22:42:05,380][base.py:123][INFO][1556] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:42:05 CST)" (scheduled at 2020-08-22 22:42:05.376505+08:00)
[2020-08-22 22:42:05,381][base.py:1020][DEBUG][12232] - Next wakeup is due at 2020-08-22 22:43:05.376505+08:00 (in 59.997091 seconds)
[2020-08-22 22:42:05,382][base.py:144][INFO][1556] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:43:05 CST)" executed successfully
[2020-08-22 22:42:44,164][base.py:440][INFO][444] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:42:44,166][base.py:881][INFO][444] - Added job "time_task" to job store "default"
[2020-08-22 22:42:44,166][base.py:166][INFO][444] - Scheduler started
[2020-08-22 22:42:44,167][base.py:940][DEBUG][11832] - Looking for jobs to run
[2020-08-22 22:42:44,168][base.py:1020][DEBUG][11832] - Next wakeup is due at 2020-08-22 22:43:44.163035+08:00 (in 59.995016 seconds)
[2020-08-22 22:42:44,222][_internal.py:113][INFO][444] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:42:56,485][_internal.py:113][INFO][13836] - 127.0.0.1 - - [22/Aug/2020 22:42:56] "[31m[1mPOST /v1/user/getToken HTTP/1.1[0m" 400 -
[2020-08-22 22:43:05,441][base.py:440][INFO][10108] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:43:05,443][base.py:881][INFO][10108] - Added job "time_task" to job store "default"
[2020-08-22 22:43:05,443][base.py:166][INFO][10108] - Scheduler started
[2020-08-22 22:43:05,445][base.py:940][DEBUG][14020] - Looking for jobs to run
[2020-08-22 22:43:05,446][base.py:1020][DEBUG][14020] - Next wakeup is due at 2020-08-22 22:44:05.441083+08:00 (in 59.996004 seconds)
[2020-08-22 22:43:05,498][_internal.py:113][INFO][10108] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:43:06,987][_internal.py:113][INFO][10452] - 127.0.0.1 - - [22/Aug/2020 22:43:06] "[31m[1mPOST /v1/user/getToken HTTP/1.1[0m" 400 -
[2020-08-22 22:44:05,457][base.py:940][DEBUG][14020] - Looking for jobs to run
[2020-08-22 22:44:05,462][base.py:123][INFO][10536] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:44:05 CST)" (scheduled at 2020-08-22 22:44:05.441083+08:00)
[2020-08-22 22:44:05,465][base.py:144][INFO][10536] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:44:05 CST)" executed successfully
[2020-08-22 22:44:05,468][base.py:1020][DEBUG][14020] - Next wakeup is due at 2020-08-22 22:45:05.441083+08:00 (in 59.982596 seconds)
[2020-08-22 22:45:05,452][base.py:940][DEBUG][14020] - Looking for jobs to run
[2020-08-22 22:45:05,455][base.py:123][INFO][10536] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:45:05 CST)" (scheduled at 2020-08-22 22:45:05.441083+08:00)
[2020-08-22 22:45:05,457][base.py:144][INFO][10536] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:45:05 CST)" executed successfully
[2020-08-22 22:45:05,461][base.py:1020][DEBUG][14020] - Next wakeup is due at 2020-08-22 22:46:05.441083+08:00 (in 59.987307 seconds)
[2020-08-22 22:46:05,463][base.py:940][DEBUG][14020] - Looking for jobs to run
[2020-08-22 22:46:05,463][base.py:123][INFO][10536] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:46:05 CST)" (scheduled at 2020-08-22 22:46:05.441083+08:00)
[2020-08-22 22:46:05,464][base.py:144][INFO][10536] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:46:05 CST)" executed successfully
[2020-08-22 22:46:05,465][base.py:1020][DEBUG][14020] - Next wakeup is due at 2020-08-22 22:47:05.441083+08:00 (in 59.977412 seconds)
[2020-08-22 22:47:05,454][base.py:940][DEBUG][14020] - Looking for jobs to run
[2020-08-22 22:47:05,455][base.py:123][INFO][14144] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:47:05 CST)" (scheduled at 2020-08-22 22:47:05.441083+08:00)
[2020-08-22 22:47:05,456][base.py:144][INFO][14144] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:47:05 CST)" executed successfully
[2020-08-22 22:47:05,457][base.py:1020][DEBUG][14020] - Next wakeup is due at 2020-08-22 22:48:05.441083+08:00 (in 59.986951 seconds)
[2020-08-22 22:48:05,447][base.py:940][DEBUG][14020] - Looking for jobs to run
[2020-08-22 22:48:05,451][base.py:123][INFO][6956] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:48:05 CST)" (scheduled at 2020-08-22 22:48:05.441083+08:00)
[2020-08-22 22:48:05,456][base.py:144][INFO][6956] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:48:05 CST)" executed successfully
[2020-08-22 22:48:05,463][base.py:1020][DEBUG][14020] - Next wakeup is due at 2020-08-22 22:49:05.441083+08:00 (in 59.992187 seconds)
[2020-08-22 22:49:05,470][base.py:940][DEBUG][14020] - Looking for jobs to run
[2020-08-22 22:49:05,475][base.py:123][INFO][10536] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:49:05 CST)" (scheduled at 2020-08-22 22:49:05.441083+08:00)
[2020-08-22 22:49:05,478][base.py:144][INFO][10536] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:49:05 CST)" executed successfully
[2020-08-22 22:49:05,483][base.py:1020][DEBUG][14020] - Next wakeup is due at 2020-08-22 22:50:05.441083+08:00 (in 59.968976 seconds)
[2020-08-22 22:49:42,846][base.py:440][INFO][14052] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:49:42,847][base.py:881][INFO][14052] - Added job "time_task" to job store "default"
[2020-08-22 22:49:42,847][base.py:166][INFO][14052] - Scheduler started
[2020-08-22 22:49:42,848][base.py:940][DEBUG][4276] - Looking for jobs to run
[2020-08-22 22:49:42,848][base.py:1020][DEBUG][4276] - Next wakeup is due at 2020-08-22 22:50:42.846266+08:00 (in 59.998002 seconds)
[2020-08-22 22:49:42,882][_internal.py:113][INFO][14052] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:49:48,555][app.py:1892][ERROR][2220] - Exception on /v1/user/login [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 148, in decorated
    user = self.authenticate(auth, password)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_httpauth.py", line 219, in authenticate
    return self.verify_password_callback(username, client_password)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 21, in verify_token
    user_info = verify_auth_token(token)
  File "D:\Poject\python\flask-restful\application\base\token_auth.py", line 49, in verify_auth_token
    password=data['password']
KeyError: 'password'
[2020-08-22 22:49:48,558][_internal.py:113][INFO][2220] - 127.0.0.1 - - [22/Aug/2020 22:49:48] "[35m[1mGET /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:50:42,848][base.py:940][DEBUG][4276] - Looking for jobs to run
[2020-08-22 22:50:42,849][base.py:123][INFO][13108] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:50:42 CST)" (scheduled at 2020-08-22 22:50:42.846266+08:00)
[2020-08-22 22:50:42,850][base.py:1020][DEBUG][4276] - Next wakeup is due at 2020-08-22 22:51:42.846266+08:00 (in 59.998041 seconds)
[2020-08-22 22:50:42,851][base.py:144][INFO][13108] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:51:42 CST)" executed successfully
[2020-08-22 22:51:17,071][base.py:440][INFO][4456] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:51:17,073][base.py:881][INFO][4456] - Added job "time_task" to job store "default"
[2020-08-22 22:51:17,073][base.py:166][INFO][4456] - Scheduler started
[2020-08-22 22:51:17,073][base.py:940][DEBUG][7148] - Looking for jobs to run
[2020-08-22 22:51:17,073][base.py:1020][DEBUG][7148] - Next wakeup is due at 2020-08-22 22:52:17.071136+08:00 (in 59.998000 seconds)
[2020-08-22 22:51:17,102][_internal.py:113][INFO][4456] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:52:17,072][base.py:940][DEBUG][7148] - Looking for jobs to run
[2020-08-22 22:52:17,073][base.py:123][INFO][5432] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:52:17 CST)" (scheduled at 2020-08-22 22:52:17.071136+08:00)
[2020-08-22 22:52:17,074][base.py:1020][DEBUG][7148] - Next wakeup is due at 2020-08-22 22:53:17.071136+08:00 (in 59.999123 seconds)
[2020-08-22 22:52:17,074][base.py:144][INFO][5432] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:53:17 CST)" executed successfully
[2020-08-22 22:52:47,316][_internal.py:113][INFO][2328] - 127.0.0.1 - - [22/Aug/2020 22:52:47] "[31m[1mGET /v1/user/login HTTP/1.1[0m" 405 -
[2020-08-22 22:53:17,076][base.py:940][DEBUG][7148] - Looking for jobs to run
[2020-08-22 22:53:17,076][base.py:123][INFO][5432] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:53:17 CST)" (scheduled at 2020-08-22 22:53:17.071136+08:00)
[2020-08-22 22:53:17,076][base.py:144][INFO][5432] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:53:17 CST)" executed successfully
[2020-08-22 22:53:17,077][base.py:1020][DEBUG][7148] - Next wakeup is due at 2020-08-22 22:54:17.071136+08:00 (in 59.994476 seconds)
[2020-08-22 22:53:25,385][userService.py:7][INFO][7688] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000001C9B963AF08>
[2020-08-22 22:53:25,397][userService.py:12][ERROR][7688] - loginService - 错误 - cannot encode object: <UnboundField(StringField, (), {})>, of type: <class 'wtforms.fields.core.UnboundField'>
[2020-08-22 22:53:25,398][app.py:1892][ERROR][7688] - Exception on /v1/user/login [POST]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2098, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
[2020-08-22 22:53:25,399][_internal.py:113][INFO][7688] - 127.0.0.1 - - [22/Aug/2020 22:53:25] "[35m[1mPOST /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:53:51,097][base.py:440][INFO][1276] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:53:51,099][base.py:881][INFO][1276] - Added job "time_task" to job store "default"
[2020-08-22 22:53:51,099][base.py:166][INFO][1276] - Scheduler started
[2020-08-22 22:53:51,101][base.py:940][DEBUG][12280] - Looking for jobs to run
[2020-08-22 22:53:51,102][base.py:1020][DEBUG][12280] - Next wakeup is due at 2020-08-22 22:54:51.097036+08:00 (in 59.995004 seconds)
[2020-08-22 22:53:51,154][_internal.py:113][INFO][1276] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:53:52,513][userService.py:7][INFO][2412] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000001F25EA83208>
[2020-08-22 22:54:32,036][userService.py:14][ERROR][2412] - loginService - 错误 - cannot encode object: <UnboundField(StringField, (), {})>, of type: <class 'wtforms.fields.core.UnboundField'>
[2020-08-22 22:54:32,037][app.py:1892][ERROR][2412] - Exception on /v1/user/login [POST]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2098, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
[2020-08-22 22:54:32,040][_internal.py:113][INFO][2412] - 127.0.0.1 - - [22/Aug/2020 22:54:32] "[35m[1mPOST /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:54:51,110][base.py:940][DEBUG][12280] - Looking for jobs to run
[2020-08-22 22:54:51,112][base.py:123][INFO][3140] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:54:51 CST)" (scheduled at 2020-08-22 22:54:51.097036+08:00)
[2020-08-22 22:54:51,113][base.py:1020][DEBUG][12280] - Next wakeup is due at 2020-08-22 22:55:51.097036+08:00 (in 59.986622 seconds)
[2020-08-22 22:54:51,114][base.py:144][INFO][3140] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:55:51 CST)" executed successfully
[2020-08-22 22:55:07,786][base.py:440][INFO][7900] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:55:07,787][base.py:881][INFO][7900] - Added job "time_task" to job store "default"
[2020-08-22 22:55:07,787][base.py:166][INFO][7900] - Scheduler started
[2020-08-22 22:55:07,787][base.py:940][DEBUG][7092] - Looking for jobs to run
[2020-08-22 22:55:07,788][base.py:1020][DEBUG][7092] - Next wakeup is due at 2020-08-22 22:56:07.786079+08:00 (in 59.998000 seconds)
[2020-08-22 22:55:07,823][_internal.py:113][INFO][7900] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:55:11,191][base.py:440][INFO][14116] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:55:11,193][base.py:881][INFO][14116] - Added job "time_task" to job store "default"
[2020-08-22 22:55:11,193][base.py:166][INFO][14116] - Scheduler started
[2020-08-22 22:55:11,194][base.py:940][DEBUG][9688] - Looking for jobs to run
[2020-08-22 22:55:11,195][base.py:1020][DEBUG][9688] - Next wakeup is due at 2020-08-22 22:56:11.190071+08:00 (in 59.995017 seconds)
[2020-08-22 22:55:11,265][_internal.py:113][INFO][14116] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:55:12,508][userService.py:8][INFO][12652] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x0000027E8D084108>
[2020-08-22 22:55:35,186][base.py:440][INFO][13264] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:55:35,188][base.py:881][INFO][13264] - Added job "time_task" to job store "default"
[2020-08-22 22:55:35,188][base.py:166][INFO][13264] - Scheduler started
[2020-08-22 22:55:35,190][base.py:940][DEBUG][2756] - Looking for jobs to run
[2020-08-22 22:55:35,191][base.py:1020][DEBUG][2756] - Next wakeup is due at 2020-08-22 22:56:35.186101+08:00 (in 59.995001 seconds)
[2020-08-22 22:55:35,243][_internal.py:113][INFO][13264] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:55:42,211][base.py:440][INFO][10052] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:55:42,213][base.py:881][INFO][10052] - Added job "time_task" to job store "default"
[2020-08-22 22:55:42,213][base.py:166][INFO][10052] - Scheduler started
[2020-08-22 22:55:42,215][base.py:940][DEBUG][1260] - Looking for jobs to run
[2020-08-22 22:55:42,216][base.py:1020][DEBUG][1260] - Next wakeup is due at 2020-08-22 22:56:42.211278+08:00 (in 59.995004 seconds)
[2020-08-22 22:55:42,275][_internal.py:113][INFO][10052] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:55:43,079][userService.py:8][INFO][2572] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x0000021680EE3108>
[2020-08-22 22:56:02,851][base.py:440][INFO][13720] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:56:02,852][base.py:881][INFO][13720] - Added job "time_task" to job store "default"
[2020-08-22 22:56:02,852][base.py:166][INFO][13720] - Scheduler started
[2020-08-22 22:56:02,853][base.py:940][DEBUG][12456] - Looking for jobs to run
[2020-08-22 22:56:02,853][base.py:1020][DEBUG][12456] - Next wakeup is due at 2020-08-22 22:57:02.851314+08:00 (in 59.998001 seconds)
[2020-08-22 22:56:02,884][_internal.py:113][INFO][13720] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:56:05,809][userService.py:8][INFO][9944] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000002640B902808>
[2020-08-22 22:56:05,820][userService.py:16][ERROR][9944] - loginService - 错误 - cannot encode object: <UnboundField(StringField, (), {})>, of type: <class 'wtforms.fields.core.UnboundField'>
[2020-08-22 22:56:05,821][_internal.py:113][INFO][9944] - 127.0.0.1 - - [22/Aug/2020 22:56:05] "[35m[1mPOST /v1/user/login HTTP/1.1[0m" 500 -
[2020-08-22 22:56:41,262][base.py:440][INFO][4800] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:56:41,263][base.py:881][INFO][4800] - Added job "time_task" to job store "default"
[2020-08-22 22:56:41,263][base.py:166][INFO][4800] - Scheduler started
[2020-08-22 22:56:41,263][base.py:940][DEBUG][4584] - Looking for jobs to run
[2020-08-22 22:56:41,264][base.py:1020][DEBUG][4584] - Next wakeup is due at 2020-08-22 22:57:41.262989+08:00 (in 59.998001 seconds)
[2020-08-22 22:56:41,292][_internal.py:113][INFO][4800] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:56:43,114][userService.py:8][INFO][960] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000002C9FEDA3788>
[2020-08-22 22:56:43,127][_internal.py:113][INFO][960] - 127.0.0.1 - - [22/Aug/2020 22:56:43] "[37mPOST /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 22:56:49,388][base.py:440][INFO][5392] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:56:49,392][base.py:881][INFO][5392] - Added job "time_task" to job store "default"
[2020-08-22 22:56:49,393][base.py:166][INFO][5392] - Scheduler started
[2020-08-22 22:56:49,395][base.py:940][DEBUG][11560] - Looking for jobs to run
[2020-08-22 22:56:49,397][base.py:1020][DEBUG][11560] - Next wakeup is due at 2020-08-22 22:57:49.388852+08:00 (in 59.992006 seconds)
[2020-08-22 22:56:49,464][_internal.py:113][INFO][5392] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:56:49,783][userService.py:8][INFO][6736] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000001F6D76D5248>
[2020-08-22 22:57:33,234][base.py:440][INFO][12676] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:57:33,235][base.py:881][INFO][12676] - Added job "time_task" to job store "default"
[2020-08-22 22:57:33,235][base.py:166][INFO][12676] - Scheduler started
[2020-08-22 22:57:33,235][base.py:940][DEBUG][12216] - Looking for jobs to run
[2020-08-22 22:57:33,235][base.py:1020][DEBUG][12216] - Next wakeup is due at 2020-08-22 22:58:33.233101+08:00 (in 59.998002 seconds)
[2020-08-22 22:57:33,265][_internal.py:113][INFO][12676] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:58:17,876][base.py:440][INFO][3268] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 22:58:17,877][base.py:881][INFO][3268] - Added job "time_task" to job store "default"
[2020-08-22 22:58:17,877][base.py:166][INFO][3268] - Scheduler started
[2020-08-22 22:58:17,878][base.py:940][DEBUG][8756] - Looking for jobs to run
[2020-08-22 22:58:17,878][base.py:1020][DEBUG][8756] - Next wakeup is due at 2020-08-22 22:59:17.876231+08:00 (in 59.997987 seconds)
[2020-08-22 22:58:17,907][_internal.py:113][INFO][3268] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 22:59:17,876][base.py:940][DEBUG][8756] - Looking for jobs to run
[2020-08-22 22:59:17,877][base.py:123][INFO][11048] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 22:59:17 CST)" (scheduled at 2020-08-22 22:59:17.876231+08:00)
[2020-08-22 22:59:17,878][base.py:1020][DEBUG][8756] - Next wakeup is due at 2020-08-22 23:00:17.876231+08:00 (in 59.999492 seconds)
[2020-08-22 22:59:17,879][base.py:144][INFO][11048] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:00:17 CST)" executed successfully
[2020-08-22 23:00:17,881][base.py:940][DEBUG][8756] - Looking for jobs to run
[2020-08-22 23:00:17,881][base.py:123][INFO][11048] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:00:17 CST)" (scheduled at 2020-08-22 23:00:17.876231+08:00)
[2020-08-22 23:00:17,882][base.py:144][INFO][11048] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:00:17 CST)" executed successfully
[2020-08-22 23:00:17,882][base.py:1020][DEBUG][8756] - Next wakeup is due at 2020-08-22 23:01:17.876231+08:00 (in 59.994693 seconds)
[2020-08-22 23:01:17,891][base.py:940][DEBUG][8756] - Looking for jobs to run
[2020-08-22 23:01:17,892][base.py:123][INFO][11048] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:01:17 CST)" (scheduled at 2020-08-22 23:01:17.876231+08:00)
[2020-08-22 23:01:17,893][base.py:1020][DEBUG][8756] - Next wakeup is due at 2020-08-22 23:02:17.876231+08:00 (in 59.984775 seconds)
[2020-08-22 23:01:17,893][base.py:144][INFO][11048] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:02:17 CST)" executed successfully
[2020-08-22 23:02:17,883][base.py:940][DEBUG][8756] - Looking for jobs to run
[2020-08-22 23:02:17,884][base.py:123][INFO][11900] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:02:17 CST)" (scheduled at 2020-08-22 23:02:17.876231+08:00)
[2020-08-22 23:02:17,884][base.py:144][INFO][11900] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:02:17 CST)" executed successfully
[2020-08-22 23:02:17,885][base.py:1020][DEBUG][8756] - Next wakeup is due at 2020-08-22 23:03:17.876231+08:00 (in 59.992910 seconds)
[2020-08-22 23:02:50,201][base.py:440][INFO][2024] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:02:50,202][base.py:881][INFO][2024] - Added job "time_task" to job store "default"
[2020-08-22 23:02:50,202][base.py:166][INFO][2024] - Scheduler started
[2020-08-22 23:02:50,203][base.py:940][DEBUG][7020] - Looking for jobs to run
[2020-08-22 23:02:50,203][base.py:1020][DEBUG][7020] - Next wakeup is due at 2020-08-22 23:03:50.201594+08:00 (in 59.997997 seconds)
[2020-08-22 23:02:50,234][_internal.py:113][INFO][2024] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:03:50,201][base.py:940][DEBUG][7020] - Looking for jobs to run
[2020-08-22 23:03:50,201][base.py:123][INFO][11788] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:03:50 CST)" (scheduled at 2020-08-22 23:03:50.201594+08:00)
[2020-08-22 23:03:50,202][base.py:1020][DEBUG][7020] - Next wakeup is due at 2020-08-22 23:04:50.201594+08:00 (in 59.999910 seconds)
[2020-08-22 23:03:50,202][base.py:144][INFO][11788] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:04:50 CST)" executed successfully
[2020-08-22 23:04:50,203][base.py:940][DEBUG][7020] - Looking for jobs to run
[2020-08-22 23:04:50,204][base.py:123][INFO][11788] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:04:50 CST)" (scheduled at 2020-08-22 23:04:50.201594+08:00)
[2020-08-22 23:04:50,205][base.py:1020][DEBUG][7020] - Next wakeup is due at 2020-08-22 23:05:50.201594+08:00 (in 59.998452 seconds)
[2020-08-22 23:04:50,205][base.py:144][INFO][11788] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:05:50 CST)" executed successfully
[2020-08-22 23:05:50,205][base.py:940][DEBUG][7020] - Looking for jobs to run
[2020-08-22 23:05:50,205][base.py:123][INFO][13612] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:05:50 CST)" (scheduled at 2020-08-22 23:05:50.201594+08:00)
[2020-08-22 23:05:50,205][base.py:1020][DEBUG][7020] - Next wakeup is due at 2020-08-22 23:06:50.201594+08:00 (in 59.996434 seconds)
[2020-08-22 23:05:50,206][base.py:144][INFO][13612] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:06:50 CST)" executed successfully
[2020-08-22 23:06:19,483][base.py:440][INFO][4512] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:06:19,483][base.py:881][INFO][4512] - Added job "time_task" to job store "default"
[2020-08-22 23:06:19,484][base.py:166][INFO][4512] - Scheduler started
[2020-08-22 23:06:19,484][base.py:940][DEBUG][4572] - Looking for jobs to run
[2020-08-22 23:06:19,484][base.py:1020][DEBUG][4572] - Next wakeup is due at 2020-08-22 23:07:19.482303+08:00 (in 59.998001 seconds)
[2020-08-22 23:06:19,513][_internal.py:113][INFO][4512] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:07:02,481][base.py:440][INFO][11196] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:07:02,482][base.py:881][INFO][11196] - Added job "time_task" to job store "default"
[2020-08-22 23:07:02,482][base.py:166][INFO][11196] - Scheduler started
[2020-08-22 23:07:02,483][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:07:02,483][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:08:02.481718+08:00 (in 59.998000 seconds)
[2020-08-22 23:07:02,511][_internal.py:113][INFO][11196] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:08:02,482][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:08:02,483][base.py:123][INFO][8020] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:08:02 CST)" (scheduled at 2020-08-22 23:08:02.481718+08:00)
[2020-08-22 23:08:02,484][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:09:02.481718+08:00 (in 59.999086 seconds)
[2020-08-22 23:08:02,485][base.py:144][INFO][8020] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:09:02 CST)" executed successfully
[2020-08-22 23:09:02,486][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:09:02,487][base.py:123][INFO][8020] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:09:02 CST)" (scheduled at 2020-08-22 23:09:02.481718+08:00)
[2020-08-22 23:09:02,488][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:10:02.481718+08:00 (in 59.995581 seconds)
[2020-08-22 23:09:02,489][base.py:144][INFO][8020] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:10:02 CST)" executed successfully
[2020-08-22 23:10:02,484][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:10:02,485][base.py:123][INFO][4400] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:10:02 CST)" (scheduled at 2020-08-22 23:10:02.481718+08:00)
[2020-08-22 23:10:02,486][base.py:144][INFO][4400] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:10:02 CST)" executed successfully
[2020-08-22 23:10:02,486][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:11:02.481718+08:00 (in 59.997335 seconds)
[2020-08-22 23:11:02,485][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:11:02,485][base.py:123][INFO][7732] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:11:02 CST)" (scheduled at 2020-08-22 23:11:02.481718+08:00)
[2020-08-22 23:11:02,485][base.py:144][INFO][7732] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:11:02 CST)" executed successfully
[2020-08-22 23:11:02,486][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:12:02.481718+08:00 (in 59.995953 seconds)
[2020-08-22 23:12:02,483][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:12:02,484][base.py:123][INFO][4400] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:12:02 CST)" (scheduled at 2020-08-22 23:12:02.481718+08:00)
[2020-08-22 23:12:02,485][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:13:02.481718+08:00 (in 59.998413 seconds)
[2020-08-22 23:12:02,486][base.py:144][INFO][4400] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:13:02 CST)" executed successfully
[2020-08-22 23:13:02,486][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:13:02,487][base.py:123][INFO][7644] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:13:02 CST)" (scheduled at 2020-08-22 23:13:02.481718+08:00)
[2020-08-22 23:13:02,488][base.py:144][INFO][7644] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:13:02 CST)" executed successfully
[2020-08-22 23:13:02,489][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:14:02.481718+08:00 (in 59.995632 seconds)
[2020-08-22 23:14:02,485][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:14:02,486][base.py:123][INFO][8020] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:14:02 CST)" (scheduled at 2020-08-22 23:14:02.481718+08:00)
[2020-08-22 23:14:02,486][base.py:144][INFO][8020] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:14:02 CST)" executed successfully
[2020-08-22 23:14:02,487][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:15:02.481718+08:00 (in 59.996089 seconds)
[2020-08-22 23:15:02,485][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:15:02,486][base.py:123][INFO][4400] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:15:02 CST)" (scheduled at 2020-08-22 23:15:02.481718+08:00)
[2020-08-22 23:15:02,487][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:16:02.481718+08:00 (in 59.996254 seconds)
[2020-08-22 23:15:02,488][base.py:144][INFO][4400] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:16:02 CST)" executed successfully
[2020-08-22 23:16:02,484][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:16:02,484][base.py:123][INFO][7732] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:16:02 CST)" (scheduled at 2020-08-22 23:16:02.481718+08:00)
[2020-08-22 23:16:02,485][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:17:02.481718+08:00 (in 59.997240 seconds)
[2020-08-22 23:16:02,485][base.py:144][INFO][7732] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:17:02 CST)" executed successfully
[2020-08-22 23:17:02,483][base.py:940][DEBUG][4724] - Looking for jobs to run
[2020-08-22 23:17:02,483][base.py:123][INFO][14180] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:17:02 CST)" (scheduled at 2020-08-22 23:17:02.481718+08:00)
[2020-08-22 23:17:02,483][base.py:1020][DEBUG][4724] - Next wakeup is due at 2020-08-22 23:18:02.481718+08:00 (in 59.998098 seconds)
[2020-08-22 23:17:02,484][base.py:144][INFO][14180] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:18:02 CST)" executed successfully
[2020-08-22 23:17:05,698][base.py:440][INFO][12548] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:17:05,699][base.py:881][INFO][12548] - Added job "time_task" to job store "default"
[2020-08-22 23:17:05,699][base.py:166][INFO][12548] - Scheduler started
[2020-08-22 23:17:05,700][base.py:940][DEBUG][9324] - Looking for jobs to run
[2020-08-22 23:17:05,700][base.py:1020][DEBUG][9324] - Next wakeup is due at 2020-08-22 23:18:05.698901+08:00 (in 59.998001 seconds)
[2020-08-22 23:17:05,736][_internal.py:113][INFO][12548] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:17:08,567][userService.py:8][INFO][12856] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000001D195EFAD08>
[2020-08-22 23:17:08,579][_internal.py:113][INFO][12856] - 127.0.0.1 - - [22/Aug/2020 23:17:08] "[37mPOST /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 23:17:12,453][_internal.py:113][INFO][5852] - 127.0.0.1 - - [22/Aug/2020 23:17:12] "[31m[1mGET /v1/user/login HTTP/1.1[0m" 405 -
[2020-08-22 23:17:39,739][base.py:440][INFO][11048] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:17:39,740][base.py:881][INFO][11048] - Added job "time_task" to job store "default"
[2020-08-22 23:17:39,740][base.py:166][INFO][11048] - Scheduler started
[2020-08-22 23:17:39,740][base.py:940][DEBUG][1816] - Looking for jobs to run
[2020-08-22 23:17:39,741][base.py:1020][DEBUG][1816] - Next wakeup is due at 2020-08-22 23:18:39.739488+08:00 (in 59.998001 seconds)
[2020-08-22 23:17:39,771][_internal.py:113][INFO][11048] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:17:50,936][userService.py:8][INFO][12660] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000002150E5CEFC8>
[2020-08-22 23:17:50,947][_internal.py:113][INFO][12660] - 127.0.0.1 - - [22/Aug/2020 23:17:50] "[37mPOST /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 23:18:39,741][base.py:940][DEBUG][1816] - Looking for jobs to run
[2020-08-22 23:18:39,741][base.py:123][INFO][11916] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:18:39 CST)" (scheduled at 2020-08-22 23:18:39.739488+08:00)
[2020-08-22 23:18:39,741][base.py:1020][DEBUG][1816] - Next wakeup is due at 2020-08-22 23:19:39.739488+08:00 (in 59.998402 seconds)
[2020-08-22 23:18:39,742][base.py:144][INFO][11916] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:19:39 CST)" executed successfully
[2020-08-22 23:19:39,755][base.py:940][DEBUG][1816] - Looking for jobs to run
[2020-08-22 23:19:39,755][base.py:123][INFO][11916] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:19:39 CST)" (scheduled at 2020-08-22 23:19:39.739488+08:00)
[2020-08-22 23:19:39,755][base.py:144][INFO][11916] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:19:39 CST)" executed successfully
[2020-08-22 23:19:39,755][base.py:1020][DEBUG][1816] - Next wakeup is due at 2020-08-22 23:20:39.739488+08:00 (in 59.983861 seconds)
[2020-08-22 23:19:41,559][base.py:440][INFO][11388] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:19:41,560][base.py:881][INFO][11388] - Added job "time_task" to job store "default"
[2020-08-22 23:19:41,560][base.py:166][INFO][11388] - Scheduler started
[2020-08-22 23:19:41,561][base.py:940][DEBUG][3104] - Looking for jobs to run
[2020-08-22 23:19:41,561][base.py:1020][DEBUG][3104] - Next wakeup is due at 2020-08-22 23:20:41.559420+08:00 (in 59.998001 seconds)
[2020-08-22 23:19:41,592][_internal.py:113][INFO][11388] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:19:47,337][userService.py:8][INFO][5736] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000001BD48617708>
[2020-08-22 23:19:47,349][_internal.py:113][INFO][5736] - 127.0.0.1 - - [22/Aug/2020 23:19:47] "[37mPOST /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 23:20:41,560][base.py:940][DEBUG][3104] - Looking for jobs to run
[2020-08-22 23:20:41,561][base.py:123][INFO][2636] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:20:41 CST)" (scheduled at 2020-08-22 23:20:41.559420+08:00)
[2020-08-22 23:20:41,562][base.py:1020][DEBUG][3104] - Next wakeup is due at 2020-08-22 23:21:41.559420+08:00 (in 59.998662 seconds)
[2020-08-22 23:20:41,563][base.py:144][INFO][2636] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:21:41 CST)" executed successfully
[2020-08-22 23:20:51,916][userService.py:8][INFO][4212] - loginService - 成功 - Parameter:<application.v1.validators.forms.UserLoginForm object at 0x000001BD49970E48>
[2020-08-22 23:20:51,917][_internal.py:113][INFO][4212] - 127.0.0.1 - - [22/Aug/2020 23:20:51] "[37mPOST /v1/user/login HTTP/1.1[0m" 200 -
[2020-08-22 23:21:21,832][_internal.py:113][INFO][548] - 127.0.0.1 - - [22/Aug/2020 23:21:21] "[31m[1mPOST /v1/user/getToken?sercet=asdaasdasds HTTP/1.1[0m" 405 -
[2020-08-22 23:21:34,440][app.py:1892][ERROR][1304] - Exception on /v1/user/getToken [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\application\v1\controllers\userController.py", line 34, in get_Token
    if form.validate_for_api():
TypeError: validate_for_api() got an unexpected keyword argument 'methods'
[2020-08-22 23:21:34,443][_internal.py:113][INFO][1304] - 127.0.0.1 - - [22/Aug/2020 23:21:34] "[35m[1mGET /v1/user/getToken?sercet=asdaasdasds HTTP/1.1[0m" 500 -
[2020-08-22 23:21:41,563][base.py:940][DEBUG][3104] - Looking for jobs to run
[2020-08-22 23:21:41,563][base.py:123][INFO][2636] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:21:41 CST)" (scheduled at 2020-08-22 23:21:41.559420+08:00)
[2020-08-22 23:21:41,564][base.py:1020][DEBUG][3104] - Next wakeup is due at 2020-08-22 23:22:41.559420+08:00 (in 59.996381 seconds)
[2020-08-22 23:21:41,564][base.py:144][INFO][2636] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:22:41 CST)" executed successfully
[2020-08-22 23:21:52,737][base.py:440][INFO][12836] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:21:52,738][base.py:881][INFO][12836] - Added job "time_task" to job store "default"
[2020-08-22 23:21:52,738][base.py:166][INFO][12836] - Scheduler started
[2020-08-22 23:21:52,738][base.py:940][DEBUG][11548] - Looking for jobs to run
[2020-08-22 23:21:52,739][base.py:1020][DEBUG][11548] - Next wakeup is due at 2020-08-22 23:22:52.737593+08:00 (in 59.999000 seconds)
[2020-08-22 23:21:52,769][_internal.py:113][INFO][12836] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:21:54,908][base.py:19][ERROR][11132] - {'secret': ['This field is required.']}
[2020-08-22 23:21:54,910][_internal.py:113][INFO][11132] - 127.0.0.1 - - [22/Aug/2020 23:21:54] "[31m[1mGET /v1/user/getToken?sercet=asdaasdasds HTTP/1.1[0m" 400 -
[2020-08-22 23:22:12,616][app.py:1892][ERROR][10228] - Exception on /v1/user/getToken [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\application\v1\controllers\userController.py", line 36, in get_Token
    token = generate_auth_token(form.account.data,expiration)
AttributeError: 'GetTokenForm' object has no attribute 'account'
[2020-08-22 23:22:12,619][_internal.py:113][INFO][10228] - 127.0.0.1 - - [22/Aug/2020 23:22:12] "[35m[1mGET /v1/user/getToken?secret=asdaasdasds HTTP/1.1[0m" 500 -
[2020-08-22 23:22:23,284][app.py:1892][ERROR][8012] - Exception on /v1/user/getToken [GET]
Traceback (most recent call last):
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\Poject\python\flask-restful\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\Poject\python\flask-restful\application\v1\controllers\userController.py", line 36, in get_Token
    token = generate_auth_token(form.account.data,expiration)
AttributeError: 'GetTokenForm' object has no attribute 'account'
[2020-08-22 23:22:23,285][_internal.py:113][INFO][8012] - 127.0.0.1 - - [22/Aug/2020 23:22:23] "[35m[1mGET /v1/user/getToken?secret=asdaasdasds HTTP/1.1[0m" 500 -
[2020-08-22 23:22:52,739][base.py:940][DEBUG][11548] - Looking for jobs to run
[2020-08-22 23:22:52,740][base.py:123][INFO][13204] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:22:52 CST)" (scheduled at 2020-08-22 23:22:52.737593+08:00)
[2020-08-22 23:22:52,741][base.py:1020][DEBUG][11548] - Next wakeup is due at 2020-08-22 23:23:52.737593+08:00 (in 59.998106 seconds)
[2020-08-22 23:22:52,742][base.py:144][INFO][13204] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:23:52 CST)" executed successfully
[2020-08-22 23:23:37,252][base.py:440][INFO][12724] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:23:37,253][base.py:881][INFO][12724] - Added job "time_task" to job store "default"
[2020-08-22 23:23:37,253][base.py:166][INFO][12724] - Scheduler started
[2020-08-22 23:23:37,254][base.py:940][DEBUG][13380] - Looking for jobs to run
[2020-08-22 23:23:37,254][base.py:1020][DEBUG][13380] - Next wakeup is due at 2020-08-22 23:24:37.252360+08:00 (in 59.998001 seconds)
[2020-08-22 23:23:37,283][_internal.py:113][INFO][12724] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:23:40,248][base.py:440][INFO][13572] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:23:40,249][base.py:881][INFO][13572] - Added job "time_task" to job store "default"
[2020-08-22 23:23:40,249][base.py:166][INFO][13572] - Scheduler started
[2020-08-22 23:23:40,249][base.py:940][DEBUG][12400] - Looking for jobs to run
[2020-08-22 23:23:40,249][base.py:1020][DEBUG][12400] - Next wakeup is due at 2020-08-22 23:24:40.247552+08:00 (in 59.998001 seconds)
[2020-08-22 23:23:40,278][_internal.py:113][INFO][13572] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:23:47,753][base.py:440][INFO][11720] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:23:47,754][base.py:881][INFO][11720] - Added job "time_task" to job store "default"
[2020-08-22 23:23:47,754][base.py:166][INFO][11720] - Scheduler started
[2020-08-22 23:23:47,755][base.py:940][DEBUG][5368] - Looking for jobs to run
[2020-08-22 23:23:47,755][base.py:1020][DEBUG][5368] - Next wakeup is due at 2020-08-22 23:24:47.753828+08:00 (in 59.998002 seconds)
[2020-08-22 23:23:47,785][_internal.py:113][INFO][11720] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:23:51,974][base.py:440][INFO][3108] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:23:51,975][base.py:881][INFO][3108] - Added job "time_task" to job store "default"
[2020-08-22 23:23:51,975][base.py:166][INFO][3108] - Scheduler started
[2020-08-22 23:23:51,975][base.py:940][DEBUG][10140] - Looking for jobs to run
[2020-08-22 23:23:51,975][base.py:1020][DEBUG][10140] - Next wakeup is due at 2020-08-22 23:24:51.973007+08:00 (in 59.998001 seconds)
[2020-08-22 23:23:52,004][_internal.py:113][INFO][3108] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:24:03,121][_internal.py:113][INFO][2588] - 127.0.0.1 - - [22/Aug/2020 23:24:03] "[37mGET /v1/user/getToken?secret=asdaasdasds HTTP/1.1[0m" 200 -
[2020-08-22 23:24:34,641][_internal.py:113][INFO][8340] - 127.0.0.1 - - [22/Aug/2020 23:24:34] "[37mGET /v1/user/getToken?secret=asdaasdasds2 HTTP/1.1[0m" 200 -
[2020-08-22 23:24:51,974][base.py:940][DEBUG][10140] - Looking for jobs to run
[2020-08-22 23:24:51,975][base.py:123][INFO][11732] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:24:51 CST)" (scheduled at 2020-08-22 23:24:51.973007+08:00)
[2020-08-22 23:24:51,975][base.py:144][INFO][11732] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:24:51 CST)" executed successfully
[2020-08-22 23:24:51,976][base.py:1020][DEBUG][10140] - Next wakeup is due at 2020-08-22 23:25:51.973007+08:00 (in 59.998805 seconds)
[2020-08-22 23:24:56,901][base.py:440][INFO][13996] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:24:56,901][base.py:881][INFO][13996] - Added job "time_task" to job store "default"
[2020-08-22 23:24:56,902][base.py:166][INFO][13996] - Scheduler started
[2020-08-22 23:24:56,902][base.py:940][DEBUG][12892] - Looking for jobs to run
[2020-08-22 23:24:56,902][base.py:1020][DEBUG][12892] - Next wakeup is due at 2020-08-22 23:25:56.900855+08:00 (in 59.998000 seconds)
[2020-08-22 23:24:56,930][_internal.py:113][INFO][13996] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:25:56,901][base.py:940][DEBUG][12892] - Looking for jobs to run
[2020-08-22 23:25:56,901][base.py:123][INFO][1108] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:25:56 CST)" (scheduled at 2020-08-22 23:25:56.900855+08:00)
[2020-08-22 23:25:56,901][base.py:1020][DEBUG][12892] - Next wakeup is due at 2020-08-22 23:26:56.900855+08:00 (in 59.999111 seconds)
[2020-08-22 23:25:56,902][base.py:144][INFO][1108] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:26:56 CST)" executed successfully
[2020-08-22 23:26:56,903][base.py:940][DEBUG][12892] - Looking for jobs to run
[2020-08-22 23:26:56,904][base.py:123][INFO][1108] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:26:56 CST)" (scheduled at 2020-08-22 23:26:56.900855+08:00)
[2020-08-22 23:26:56,905][base.py:1020][DEBUG][12892] - Next wakeup is due at 2020-08-22 23:27:56.900855+08:00 (in 59.997181 seconds)
[2020-08-22 23:26:56,906][base.py:144][INFO][1108] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:27:56 CST)" executed successfully
[2020-08-22 23:27:49,753][base.py:440][INFO][12056] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:27:49,754][base.py:881][INFO][12056] - Added job "time_task" to job store "default"
[2020-08-22 23:27:49,755][base.py:166][INFO][12056] - Scheduler started
[2020-08-22 23:27:49,755][base.py:940][DEBUG][9180] - Looking for jobs to run
[2020-08-22 23:27:49,755][base.py:1020][DEBUG][9180] - Next wakeup is due at 2020-08-22 23:28:49.753787+08:00 (in 59.998000 seconds)
[2020-08-22 23:27:49,786][_internal.py:113][INFO][12056] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:28:11,343][base.py:440][INFO][8308] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:28:11,345][base.py:881][INFO][8308] - Added job "time_task" to job store "default"
[2020-08-22 23:28:11,345][base.py:166][INFO][8308] - Scheduler started
[2020-08-22 23:28:11,345][base.py:940][DEBUG][5376] - Looking for jobs to run
[2020-08-22 23:28:11,345][base.py:1020][DEBUG][5376] - Next wakeup is due at 2020-08-22 23:29:11.343489+08:00 (in 59.998001 seconds)
[2020-08-22 23:28:11,374][_internal.py:113][INFO][8308] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:28:14,191][userService.py:23][INFO][7260] - loginService - 成功 - Parameter:<application.v1.validators.forms.GetTokenForm object at 0x000002ECE86C48C8>
[2020-08-22 23:28:14,197][userService.py:30][ERROR][7260] - loginService - 错误 - 'GetTokenForm' object has no attribute 'account'
[2020-08-22 23:28:14,198][_internal.py:113][INFO][7260] - 127.0.0.1 - - [22/Aug/2020 23:28:14] "[35m[1mGET /v1/user/getToken?secret=asdaasdasds2 HTTP/1.1[0m" 500 -
[2020-08-22 23:29:11,345][base.py:940][DEBUG][5376] - Looking for jobs to run
[2020-08-22 23:29:11,345][base.py:123][INFO][13936] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:29:11 CST)" (scheduled at 2020-08-22 23:29:11.343489+08:00)
[2020-08-22 23:29:11,345][base.py:1020][DEBUG][5376] - Next wakeup is due at 2020-08-22 23:30:11.343489+08:00 (in 59.997686 seconds)
[2020-08-22 23:29:11,346][base.py:144][INFO][13936] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:30:11 CST)" executed successfully
[2020-08-22 23:29:19,660][base.py:440][INFO][7644] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:29:19,661][base.py:881][INFO][7644] - Added job "time_task" to job store "default"
[2020-08-22 23:29:19,661][base.py:166][INFO][7644] - Scheduler started
[2020-08-22 23:29:19,661][base.py:940][DEBUG][2764] - Looking for jobs to run
[2020-08-22 23:29:19,661][base.py:1020][DEBUG][2764] - Next wakeup is due at 2020-08-22 23:30:19.660405+08:00 (in 59.999002 seconds)
[2020-08-22 23:29:19,705][_internal.py:113][INFO][7644] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:29:20,894][userService.py:23][INFO][10692] - loginService - 成功 - Parameter:<application.v1.validators.forms.GetTokenForm object at 0x0000018546752788>
[2020-08-22 23:29:20,896][_internal.py:113][INFO][10692] - 127.0.0.1 - - [22/Aug/2020 23:29:20] "[37mGET /v1/user/getToken?secret=asdaasdasds2 HTTP/1.1[0m" 200 -
[2020-08-22 23:30:19,661][base.py:940][DEBUG][2764] - Looking for jobs to run
[2020-08-22 23:30:19,662][base.py:123][INFO][11168] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:30:19 CST)" (scheduled at 2020-08-22 23:30:19.660405+08:00)
[2020-08-22 23:30:19,662][base.py:144][INFO][11168] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:30:19 CST)" executed successfully
[2020-08-22 23:30:19,663][base.py:1020][DEBUG][2764] - Next wakeup is due at 2020-08-22 23:31:19.660405+08:00 (in 59.998673 seconds)
[2020-08-22 23:31:19,677][base.py:940][DEBUG][2764] - Looking for jobs to run
[2020-08-22 23:31:19,677][base.py:123][INFO][11168] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:31:19 CST)" (scheduled at 2020-08-22 23:31:19.660405+08:00)
[2020-08-22 23:31:19,677][base.py:144][INFO][11168] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:31:19 CST)" executed successfully
[2020-08-22 23:31:19,678][base.py:1020][DEBUG][2764] - Next wakeup is due at 2020-08-22 23:32:19.660405+08:00 (in 59.983391 seconds)
[2020-08-22 23:31:38,263][base.py:440][INFO][4792] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:31:38,264][base.py:881][INFO][4792] - Added job "time_task" to job store "default"
[2020-08-22 23:31:38,264][base.py:166][INFO][4792] - Scheduler started
[2020-08-22 23:31:38,265][base.py:940][DEBUG][11716] - Looking for jobs to run
[2020-08-22 23:31:38,265][base.py:1020][DEBUG][11716] - Next wakeup is due at 2020-08-22 23:32:38.263755+08:00 (in 59.998007 seconds)
[2020-08-22 23:31:38,297][_internal.py:113][INFO][4792] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:31:40,780][userService.py:24][INFO][14120] - getToken - 成功 - Parameter:<application.v1.validators.forms.GetTokenForm object at 0x0000025FA4F038C8>
[2020-08-22 23:31:40,781][_internal.py:113][INFO][14120] - 127.0.0.1 - - [22/Aug/2020 23:31:40] "[37mGET /v1/user/getToken?secret=asdaasdasds2 HTTP/1.1[0m" 200 -
[2020-08-22 23:31:45,274][userService.py:24][INFO][13684] - getToken - 成功 - Parameter:<application.v1.validators.forms.GetTokenForm object at 0x0000025FA4F15548>
[2020-08-22 23:31:45,275][_internal.py:113][INFO][13684] - 127.0.0.1 - - [22/Aug/2020 23:31:45] "[37mGET /v1/user/getToken?secret=asdaasdasds22 HTTP/1.1[0m" 200 -
[2020-08-22 23:32:38,265][base.py:940][DEBUG][11716] - Looking for jobs to run
[2020-08-22 23:32:38,265][base.py:123][INFO][12776] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:32:38 CST)" (scheduled at 2020-08-22 23:32:38.263755+08:00)
[2020-08-22 23:32:38,266][base.py:1020][DEBUG][11716] - Next wakeup is due at 2020-08-22 23:33:38.263755+08:00 (in 59.998616 seconds)
[2020-08-22 23:32:38,266][base.py:144][INFO][12776] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:33:38 CST)" executed successfully
[2020-08-22 23:33:19,272][base.py:440][INFO][8440] - Adding job tentatively -- it will be properly scheduled when the scheduler starts
[2020-08-22 23:33:19,274][base.py:881][INFO][8440] - Added job "time_task" to job store "default"
[2020-08-22 23:33:19,274][base.py:166][INFO][8440] - Scheduler started
[2020-08-22 23:33:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:33:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:34:19.272724+08:00 (in 59.998001 seconds)
[2020-08-22 23:33:19,315][_internal.py:113][INFO][8440] -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
[2020-08-22 23:33:21,633][userService.py:24][INFO][4352] - getToken - 成功 - Parameter:<application.v1.validators.forms.GetTokenForm object at 0x0000017F2F014788>
[2020-08-22 23:33:21,655][_internal.py:113][INFO][4352] - 127.0.0.1 - - [22/Aug/2020 23:33:21] "[37mGET /v1/user/getToken?secret=asdaasdasds22 HTTP/1.1[0m" 200 -
[2020-08-22 23:33:26,126][userService.py:24][INFO][6956] - getToken - 成功 - Parameter:<application.v1.validators.forms.GetTokenForm object at 0x0000017F2F02BF88>
[2020-08-22 23:33:26,129][_internal.py:113][INFO][6956] - 127.0.0.1 - - [22/Aug/2020 23:33:26] "[37mGET /v1/user/getToken?secret=asdaasdasds HTTP/1.1[0m" 200 -
[2020-08-22 23:34:19,275][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:34:19,276][base.py:123][INFO][11436] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:34:19 CST)" (scheduled at 2020-08-22 23:34:19.272724+08:00)
[2020-08-22 23:34:19,277][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:35:19.272724+08:00 (in 59.997571 seconds)
[2020-08-22 23:34:19,277][base.py:144][INFO][11436] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:35:19 CST)" executed successfully
[2020-08-22 23:35:19,276][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:35:19,276][base.py:123][INFO][11436] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:35:19 CST)" (scheduled at 2020-08-22 23:35:19.272724+08:00)
[2020-08-22 23:35:19,277][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:36:19.272724+08:00 (in 59.996000 seconds)
[2020-08-22 23:35:19,277][base.py:144][INFO][11436] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:36:19 CST)" executed successfully
[2020-08-22 23:36:17,900][_internal.py:113][INFO][7028] - 127.0.0.1 - - [22/Aug/2020 23:36:17] "[31m[1mGET /v1/user/test123 HTTP/1.1[0m" 401 -
[2020-08-22 23:36:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:36:19,274][base.py:123][INFO][13160] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:36:19 CST)" (scheduled at 2020-08-22 23:36:19.272724+08:00)
[2020-08-22 23:36:19,275][base.py:144][INFO][13160] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:36:19 CST)" executed successfully
[2020-08-22 23:36:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:37:19.272724+08:00 (in 59.998688 seconds)
[2020-08-22 23:37:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:37:19,275][base.py:123][INFO][11436] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:37:19 CST)" (scheduled at 2020-08-22 23:37:19.272724+08:00)
[2020-08-22 23:37:19,276][base.py:144][INFO][11436] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:37:19 CST)" executed successfully
[2020-08-22 23:37:19,277][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:38:19.272724+08:00 (in 59.998159 seconds)
[2020-08-22 23:38:19,277][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:38:19,278][base.py:123][INFO][13160] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:38:19 CST)" (scheduled at 2020-08-22 23:38:19.272724+08:00)
[2020-08-22 23:38:19,278][base.py:144][INFO][13160] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:38:19 CST)" executed successfully
[2020-08-22 23:38:19,279][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:39:19.272724+08:00 (in 59.995104 seconds)
[2020-08-22 23:39:19,276][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:39:19,277][base.py:123][INFO][5148] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:39:19 CST)" (scheduled at 2020-08-22 23:39:19.272724+08:00)
[2020-08-22 23:39:19,277][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:40:19.272724+08:00 (in 59.996384 seconds)
[2020-08-22 23:39:19,278][base.py:144][INFO][5148] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:40:19 CST)" executed successfully
[2020-08-22 23:40:19,275][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:40:19,276][base.py:123][INFO][13988] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:40:19 CST)" (scheduled at 2020-08-22 23:40:19.272724+08:00)
[2020-08-22 23:40:19,276][base.py:144][INFO][13988] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:40:19 CST)" executed successfully
[2020-08-22 23:40:19,277][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:41:19.272724+08:00 (in 59.997226 seconds)
[2020-08-22 23:41:19,275][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:41:19,276][base.py:123][INFO][13160] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:41:19 CST)" (scheduled at 2020-08-22 23:41:19.272724+08:00)
[2020-08-22 23:41:19,277][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:42:19.272724+08:00 (in 59.997078 seconds)
[2020-08-22 23:41:19,278][base.py:144][INFO][13160] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:42:19 CST)" executed successfully
[2020-08-22 23:42:19,276][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:42:19,277][base.py:123][INFO][13180] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:42:19 CST)" (scheduled at 2020-08-22 23:42:19.272724+08:00)
[2020-08-22 23:42:19,278][base.py:144][INFO][13180] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:42:19 CST)" executed successfully
[2020-08-22 23:42:19,279][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:43:19.272724+08:00 (in 59.995822 seconds)
[2020-08-22 23:43:19,276][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:43:19,277][base.py:123][INFO][13988] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:43:19 CST)" (scheduled at 2020-08-22 23:43:19.272724+08:00)
[2020-08-22 23:43:19,278][base.py:144][INFO][13988] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:43:19 CST)" executed successfully
[2020-08-22 23:43:19,279][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:44:19.272724+08:00 (in 59.996503 seconds)
[2020-08-22 23:44:19,276][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:44:19,277][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:45:19.272724+08:00 (in 59.995956 seconds)
[2020-08-22 23:44:19,277][base.py:123][INFO][10308] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:45:19 CST)" (scheduled at 2020-08-22 23:44:19.272724+08:00)
[2020-08-22 23:44:19,278][base.py:144][INFO][10308] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:45:19 CST)" executed successfully
[2020-08-22 23:45:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:45:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:46:19.272724+08:00 (in 59.998711 seconds)
[2020-08-22 23:45:19,274][base.py:123][INFO][4268] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:46:19 CST)" (scheduled at 2020-08-22 23:45:19.272724+08:00)
[2020-08-22 23:45:19,276][base.py:144][INFO][4268] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:46:19 CST)" executed successfully
[2020-08-22 23:46:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:46:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:47:19.272724+08:00 (in 59.998077 seconds)
[2020-08-22 23:46:19,275][base.py:123][INFO][13180] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:47:19 CST)" (scheduled at 2020-08-22 23:46:19.272724+08:00)
[2020-08-22 23:46:19,275][base.py:144][INFO][13180] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:47:19 CST)" executed successfully
[2020-08-22 23:47:19,275][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:47:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:48:19.272724+08:00 (in 59.997177 seconds)
[2020-08-22 23:47:19,276][base.py:123][INFO][11768] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:48:19 CST)" (scheduled at 2020-08-22 23:47:19.272724+08:00)
[2020-08-22 23:47:19,277][base.py:144][INFO][11768] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:48:19 CST)" executed successfully
[2020-08-22 23:48:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:48:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:49:19.272724+08:00 (in 59.998095 seconds)
[2020-08-22 23:48:19,275][base.py:123][INFO][13860] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:49:19 CST)" (scheduled at 2020-08-22 23:48:19.272724+08:00)
[2020-08-22 23:48:19,276][base.py:144][INFO][13860] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:49:19 CST)" executed successfully
[2020-08-22 23:49:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:49:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:50:19.272724+08:00 (in 59.997838 seconds)
[2020-08-22 23:49:19,275][base.py:123][INFO][10308] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:50:19 CST)" (scheduled at 2020-08-22 23:49:19.272724+08:00)
[2020-08-22 23:49:19,275][base.py:144][INFO][10308] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:50:19 CST)" executed successfully
[2020-08-22 23:50:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:50:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:51:19.272724+08:00 (in 59.998653 seconds)
[2020-08-22 23:50:19,275][base.py:123][INFO][4268] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:51:19 CST)" (scheduled at 2020-08-22 23:50:19.272724+08:00)
[2020-08-22 23:50:19,276][base.py:144][INFO][4268] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:51:19 CST)" executed successfully
[2020-08-22 23:51:19,275][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:51:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:52:19.272724+08:00 (in 59.997721 seconds)
[2020-08-22 23:51:19,275][base.py:123][INFO][5148] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:52:19 CST)" (scheduled at 2020-08-22 23:51:19.272724+08:00)
[2020-08-22 23:51:19,276][base.py:144][INFO][5148] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:52:19 CST)" executed successfully
[2020-08-22 23:52:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:52:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:53:19.272724+08:00 (in 59.998045 seconds)
[2020-08-22 23:52:19,275][base.py:123][INFO][13988] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:53:19 CST)" (scheduled at 2020-08-22 23:52:19.272724+08:00)
[2020-08-22 23:52:19,276][base.py:144][INFO][13988] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:53:19 CST)" executed successfully
[2020-08-22 23:53:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:53:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:54:19.272724+08:00 (in 59.997818 seconds)
[2020-08-22 23:53:19,275][base.py:123][INFO][13860] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:54:19 CST)" (scheduled at 2020-08-22 23:53:19.272724+08:00)
[2020-08-22 23:53:19,276][base.py:144][INFO][13860] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:54:19 CST)" executed successfully
[2020-08-22 23:54:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:54:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:55:19.272724+08:00 (in 59.997964 seconds)
[2020-08-22 23:54:19,274][base.py:123][INFO][10308] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:55:19 CST)" (scheduled at 2020-08-22 23:54:19.272724+08:00)
[2020-08-22 23:54:19,275][base.py:144][INFO][10308] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:55:19 CST)" executed successfully
[2020-08-22 23:55:19,273][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:55:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:56:19.272724+08:00 (in 59.999097 seconds)
[2020-08-22 23:55:19,274][base.py:123][INFO][13160] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:56:19 CST)" (scheduled at 2020-08-22 23:55:19.272724+08:00)
[2020-08-22 23:55:19,275][base.py:144][INFO][13160] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:56:19 CST)" executed successfully
[2020-08-22 23:56:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:56:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:57:19.272724+08:00 (in 59.997855 seconds)
[2020-08-22 23:56:19,275][base.py:123][INFO][13180] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:57:19 CST)" (scheduled at 2020-08-22 23:56:19.272724+08:00)
[2020-08-22 23:56:19,276][base.py:144][INFO][13180] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:57:19 CST)" executed successfully
[2020-08-22 23:57:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:57:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:58:19.272724+08:00 (in 59.998553 seconds)
[2020-08-22 23:57:19,275][base.py:123][INFO][13988] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:58:19 CST)" (scheduled at 2020-08-22 23:57:19.272724+08:00)
[2020-08-22 23:57:19,276][base.py:144][INFO][13988] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:58:19 CST)" executed successfully
[2020-08-22 23:58:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:58:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-22 23:59:19.272724+08:00 (in 59.998437 seconds)
[2020-08-22 23:58:19,275][base.py:123][INFO][7396] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:59:19 CST)" (scheduled at 2020-08-22 23:58:19.272724+08:00)
[2020-08-22 23:58:19,275][base.py:144][INFO][7396] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-22 23:59:19 CST)" executed successfully
[2020-08-22 23:59:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-22 23:59:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-23 00:00:19.272724+08:00 (in 59.998434 seconds)
[2020-08-22 23:59:19,275][base.py:123][INFO][10308] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:00:19 CST)" (scheduled at 2020-08-22 23:59:19.272724+08:00)
[2020-08-22 23:59:19,276][base.py:144][INFO][10308] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:00:19 CST)" executed successfully
[2020-08-23 00:00:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-23 00:00:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-23 00:01:19.272724+08:00 (in 59.997875 seconds)
[2020-08-23 00:00:19,275][base.py:123][INFO][4268] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:01:19 CST)" (scheduled at 2020-08-23 00:00:19.272724+08:00)
[2020-08-23 00:00:19,276][base.py:144][INFO][4268] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:01:19 CST)" executed successfully
[2020-08-23 00:01:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-23 00:01:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-23 00:02:19.272724+08:00 (in 59.997995 seconds)
[2020-08-23 00:01:19,275][base.py:123][INFO][5148] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:02:19 CST)" (scheduled at 2020-08-23 00:01:19.272724+08:00)
[2020-08-23 00:01:19,276][base.py:144][INFO][5148] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:02:19 CST)" executed successfully
[2020-08-23 00:02:19,273][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-23 00:02:19,274][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-23 00:03:19.272724+08:00 (in 59.998778 seconds)
[2020-08-23 00:02:19,274][base.py:123][INFO][13988] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:03:19 CST)" (scheduled at 2020-08-23 00:02:19.272724+08:00)
[2020-08-23 00:02:19,275][base.py:144][INFO][13988] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:03:19 CST)" executed successfully
[2020-08-23 00:03:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-23 00:03:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-23 00:04:19.272724+08:00 (in 59.997758 seconds)
[2020-08-23 00:03:19,275][base.py:123][INFO][7396] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:04:19 CST)" (scheduled at 2020-08-23 00:03:19.272724+08:00)
[2020-08-23 00:03:19,276][base.py:144][INFO][7396] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:04:19 CST)" executed successfully
[2020-08-23 00:04:19,274][base.py:940][DEBUG][13248] - Looking for jobs to run
[2020-08-23 00:04:19,275][base.py:1020][DEBUG][13248] - Next wakeup is due at 2020-08-23 00:05:19.272724+08:00 (in 59.998061 seconds)
[2020-08-23 00:04:19,275][base.py:123][INFO][11436] - Running job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:05:19 CST)" (scheduled at 2020-08-23 00:04:19.272724+08:00)
[2020-08-23 00:04:19,276][base.py:144][INFO][11436] - Job "time_task (trigger: interval[0:01:00], next run at: 2020-08-23 00:05:19 CST)" executed successfully
